% Options for packages loaded elsewhere
% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
  11pt,
]{article}
\usepackage{xcolor}
\usepackage[margin=1.0in]{geometry}
\usepackage{amsmath,amssymb}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage{setspace}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
% Make \paragraph and \subparagraph free-standing
\makeatletter
\ifx\paragraph\undefined\else
  \let\oldparagraph\paragraph
  \renewcommand{\paragraph}{
    \@ifstar
      \xxxParagraphStar
      \xxxParagraphNoStar
  }
  \newcommand{\xxxParagraphStar}[1]{\oldparagraph*{#1}\mbox{}}
  \newcommand{\xxxParagraphNoStar}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
  \let\oldsubparagraph\subparagraph
  \renewcommand{\subparagraph}{
    \@ifstar
      \xxxSubParagraphStar
      \xxxSubParagraphNoStar
  }
  \newcommand{\xxxSubParagraphStar}[1]{\oldsubparagraph*{#1}\mbox{}}
  \newcommand{\xxxSubParagraphNoStar}[1]{\oldsubparagraph{#1}\mbox{}}
\fi
\makeatother


\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\newsavebox\pandoc@box
\newcommand*\pandocbounded[1]{% scales image to fit in text height/width
  \sbox\pandoc@box{#1}%
  \Gscale@div\@tempa{\textheight}{\dimexpr\ht\pandoc@box+\dp\pandoc@box\relax}%
  \Gscale@div\@tempb{\linewidth}{\wd\pandoc@box}%
  \ifdim\@tempb\p@<\@tempa\p@\let\@tempa\@tempb\fi% select the smaller of both
  \ifdim\@tempa\p@<\p@\scalebox{\@tempa}{\usebox\pandoc@box}%
  \else\usebox{\pandoc@box}%
  \fi%
}
% Set default figure placement to htbp
\def\fps@figure{htbp}
\makeatother


% definitions for citeproc citations
\NewDocumentCommand\citeproctext{}{}
\NewDocumentCommand\citeproc{mm}{%
  \begingroup\def\citeproctext{#2}\cite{#1}\endgroup}
\makeatletter
 % allow citations to break across lines
 \let\@cite@ofmt\@firstofone
 % avoid brackets around text for \cite:
 \def\@biblabel#1{}
 \def\@cite#1#2{{#1\if@tempswa , #2\fi}}
\makeatother
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[2] % #1 hanging-indent, #2 entry-spacing
 {\begin{list}{}{%
  \setlength{\itemindent}{0pt}
  \setlength{\leftmargin}{0pt}
  \setlength{\parsep}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
   \setlength{\leftmargin}{\cslhangindent}
   \setlength{\itemindent}{-1\cslhangindent}
  \fi
  % set entry spacing
  \setlength{\itemsep}{#2\baselineskip}}}
 {\end{list}}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{\hfill\break\parbox[t]{\linewidth}{\strut\ignorespaces#1\strut}}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{\strut#1\strut}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{\strut#1\strut}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}



\setlength{\emergencystretch}{3em} % prevent overfull lines

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}



 


\usepackage{booktabs}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{xcolor}
\usepackage[left]{lineno}
\linenumbers
\modulolinenumbers
\usepackage{helvet}
\renewcommand*\familydefault{\sfdefault}
\usepackage[T1]{fontenc}
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\AtBeginDocument{%
\ifdefined\contentsname
  \renewcommand*\contentsname{Table of contents}
\else
  \newcommand\contentsname{Table of contents}
\fi
\ifdefined\listfigurename
  \renewcommand*\listfigurename{List of Figures}
\else
  \newcommand\listfigurename{List of Figures}
\fi
\ifdefined\listtablename
  \renewcommand*\listtablename{List of Tables}
\else
  \newcommand\listtablename{List of Tables}
\fi
\ifdefined\figurename
  \renewcommand*\figurename{Figure}
\else
  \newcommand\figurename{Figure}
\fi
\ifdefined\tablename
  \renewcommand*\tablename{Table}
\else
  \newcommand\tablename{Table}
\fi
}
\@ifpackageloaded{float}{}{\usepackage{float}}
\floatstyle{ruled}
\@ifundefined{c@chapter}{\newfloat{codelisting}{h}{lop}}{\newfloat{codelisting}{h}{lop}[chapter]}
\floatname{codelisting}{Listing}
\newcommand*\listoflistings{\listof{codelisting}{List of Listings}}
\makeatother
\makeatletter
\makeatother
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\@ifpackageloaded{subcaption}{}{\usepackage{subcaption}}
\makeatother
\usepackage{bookmark}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Data Accessibility Paper},
  pdfauthor={Joanna Colovas; Adena Collens; Patrick D. Schloss},
  pdfkeywords={data accessibility, data reproducibility, supervised
machine learning},
  colorlinks=true,
  linkcolor={blue},
  filecolor={Maroon},
  citecolor={Blue},
  urlcolor={Blue},
  pdfcreator={LaTeX via pandoc}}


\title{Data Accessibility Paper}
\author{Joanna Colovas \and Adena Collens \and Patrick D. Schloss}
\date{Nov 11, 2025}
\begin{document}
\maketitle


\setstretch{1.75}
\begin{itemize}
\item
  Abstract
\item
  Importance

  \begin{itemize}
  \tightlist
  \item
    Incentivize authors to publish/make available their original data
  \item
    Publishing data helps get more use out of research
  \item
    Helps eliminate file drawer effect as it shows negative data
  \end{itemize}
\item
  Keywords

  \begin{itemize}
  \tightlist
  \item
    Data accessibility
  \item
    Data reproducibility
  \end{itemize}
\end{itemize}

\subsection{Introduction}\label{introduction}

\subsubsection{Scientific Data as a Public
Good}\label{scientific-data-as-a-public-good}

The United States Government spent over two hundred million dollars
(USD) in 2024 on research expenditures ((1)). The result of all of these
investments were data, paid for by taxpayers. Therefore, data are a
public good. Public goods, for example public libraries, are able to be
used by anyone without barrier to entry, and without diminishing the use
of others. Data are best used for the benefit of those who provided the
funds for it. Generated data are useful in many ways, not only by the
original generators to analyze and answer study questions, but further
used to answer additional questions on the same study system, to
replicate the original analyses, and for meta-studies, by combining
multiple similar datasets. A key tenet of the scientific method is this
ability to replicate scientific findings to ensure that they are not due
to error. Scientific findings can be replicated by re-completing the
same analyses by another researcher, or by completing another type of
analysis on the same data. This is only possible when the data used to
complete the original analyses are available for use. Additionally, data
are used to eliminate possible solutions to a problem by the publishing
of negative or non-significant data. Thinking of negative data as a
public good, their availability help researchers avoid sinking time and
financial resources into investigation of non-viable hypotheses. Funding
sources are not available to pursue the generation of negative data, and
agencies look to support fruitful research. As a result, researchers
have few incentives to publish negative or non-significant results. This
lack of publication of non-fruitful investigation is more commonly known
as the ``file drawer effect'' ((2), (3)). In this current time period
when government funding is uncertain, it is more important than ever to
pursue fruitful research.

With the latest and greatest methodologies available across fields,
increasing amounts of data are being generated each day, especially in
the biological sciences where large and complex datasets are the new
standard. ( (4), (5)). Availability of large quantities of study data
and their associated metadata (data about data) are necessary resources
for appropriate use and re-use of data, protocols, as well as recreation
of analyses. Data availability are a deeply important component of the
scientific process in the digital age, and the curation of digital
records is a slowly emerging topic in data science( (6)). Available data
and analyses are the gold standard for recreation of studies and
replication of their results. Not only is replication a worthy goal, but
large datasets are often underutilized, and can continue to provide
benefit and resources to researchers via their re-use towards
investigating and answering further questions. As a result, the National
Institutes of Health (NIH) has called for grant proposals for the
creation, enhancement, and maintenance, of new and existing data
repositories ((7), (8)).

There are three major databases worldwide to support sequencing and
sharing efforts. The National Library of Medicine's (NLM) National
Center for Biotechnology (NCBI) in the United States, the Research
Organization of Information Systems' (ROIS) National Institute of
Genetics (NIG) in Japan, and the European Molecular Biology Lab's (EMBL)
European Bioinformatics Institue (EBI) in Europe. These three databases
are part of the International Nucleotide Sequence Database Collaboration
(INSDC) ((9)). Comparative genetics and genomics would not be possible
without strong community commitment to data availability.

\subsubsection{American Society for Microbiology Journals
(ASM)}\label{american-society-for-microbiology-journals-asm}

The American Society for Microbiology(ASM) is the major professional
body recognized by microbiologists. They have eighteen journals,
thirteen primary research journals, three review journals, and two
archive journals. In addition, several journals have been folded into
others or renamed over time. In this study, we considered twelve of
these journals, \emph{Applied and Envrionmental Microbiology,
Antimicrobial Agents and Chemotherapy, Infection and Immunity, Journal
of Clinical Biology, Journal of Virology, Journal of Bacteriology,
Journal of Microbiology and Biology Education, Microbiology Resource
Announcments} (formerly known as \emph{Genome Announcements}),
\emph{mSystems, mSphere, mBio, and Microbiology Spectrum.} Of note,
several journals had changes to their publication goals during the
2000-2024 time period. The \emph{Journal of Bacteriology} was the
primary journal to publish new genome announcements until 2013 when ASM
announced journal \emph{Genome Announcements} as a more permanent
destination for this type of data. \emph{Genome Announcements} was
active from 2013 until 2018, when it was re-branded to
\emph{Microbiology Resource Announcements}, which has been active from
2018 until present. Another journal of note was \emph{Microbiology
Spectrum} and its re-brand. From 2013 until the fall of 2021,
\emph{Microbiology Spectrum} was a review journal. At this point and
beyond, \emph{Microbiology Spectrum} became a primary research journal
((10)). Several journals, \emph{mBio} (b.2010), \emph{Microbiology
Spectrum} (b. 2013, re-brand 2021), \emph{mSphere} (b. 2016),
\emph{mSystems} (b. 2016), and \emph{Genome Announcements} (2013-2018)
all did not span the entire study period of interest.

\subsubsection{Current Data Availability
Policies}\label{current-data-availability-policies}

Current data availability guidelines have been informed by a number of
policies created by funding agencies, peer-review journals, conference
and special task groups, as well as community interest groups. In 2011,
after the Future of Research Communication (FoRC) conference in Germany,
scientists and others came together to establish FORCE11, a community
interest group which sought to encourage and promote data availability
standards ((11)). In 2014 the FORCE11 group published the Joint
Declaration of Data Citation Principles (JDDCP), a document with
continued work towards the standardization of data citation and future
availability((12)). Some of the JDDCPs included crediting the authors of
the data, providing data with unique identifiers, and the persistence of
available data.

Also in 2011, the Genomic Standards Consortium (GSC) published a set of
standards in \emph{Nature Biotechnology} to promote the publication of
the ``minimum information about a marker gene sequence'' (MIMARKS) or
``minimum information about x sequence'' (MIxS) ((13)). These standards
are checklists usable by data generators and uploaders towards inclusion
of relevant data with sequence uploads in the International Nucleotide
Sequence Database Collaboration (INSDC). Some checklist items include if
the data were published to an \textbf{INSDC} database and metadata about
the study systems, data collected, and authors. An important factor was
the ability to link the data to the results and to the data generators.
The Findable, Accessible, Interoperable, and Reuseable (FAIR) data
science guiding principles that were put forth in 2016 in \emph{Nature
Scientific Data} urges readers to ``improve the infrastructure
supporting the reuse of scholarly data'' ((14)). The FAIR principles
were often cited by NIH in funding calls for strong data science
practices((7)).

In 2021, a \emph{Nature Medicine} publication put forth the
``Strengthening the Organization and Reporting of Microbiome Studies''
(STORMS) checklist to help authors self-identification of report-worthy
elements of their data and metadata ((15)). Some items on the STORMS
checklist included reporting the sequencing method used in the study,
the study design, and physical location of the study. Unfortunately,
none of these data availability principles or checklists were yet
enforceable by any agency.

The National Institutes of Health (NIH) began enforcing the ``Policy for
Data Management and Sharing'' (NOT-OD-21-013) in January of 2023,
requiring all NIH funded studies to submit a data management and sharing
plan (DMS) with their funding applications, and comply with their DMS
plan after generation and publication of the funded work ((16)). A DMS
plan includes detailed descriptions of data that will be generated in a
study, related tools, standards, and data preservation plans.
Non-compliance with NOT-OD-21-013 is identified by funding agencies
during annual Research Performance Progress Reports (RPPRs), and may
impact future funding decisions ((16)). The NIH policy for
non-compliance with award terms and conditions varies due to the type of
research misconduct, but is clear that the NIH will protect their own
interests, including placing conditions on awards, preventing future
awards, or closer monitoring of award activities. In this time of
uncertain funding from the NIH and other funding agencies, it is more
important that ever for investigators to maintain continued compliance.
This compliance starts with readily available data and manuscripts.

At time of publication, the ASM journal program required that authors
``make data fully available, without restriction, except in rare
circumstances'' ((17)). They have adapted this policy from journals
\emph{Microbial Genomics} and \emph{PLOS}. In the ASM open data policy
they described the use of a ``Data Availability Statement'' which
includes ``data description, name(s) of the repositories, and digital
object identifiers (DOIs) or accession numbers'' and encouraged
publishing data on relevant public repositories ((17)). Consequences of
non-compliance to the ASM open data policy included contacting research
article authors to inform of non-compliance, publication of an
``Expression of Concern'' for the author and their compliance issues,
future sanctions on publication in ASM journals, as well as contacting
the affiliated research institution and/or funding agencies of the
authors ((18)). We endeavored to evaluate how well the microbiology
community is using reproducible data practices as we believed that this
group of researchers were early adopters of technologies available as a
result of both the ASM and NIH policies towards data availability.

\subsubsection{Historical Nucleic Acid Sequencing Efforts and
Examples}\label{historical-nucleic-acid-sequencing-efforts-and-examples}

Beginning in 1996 with the International Strategy Meeting on Human
Genome Sequencing in Bermuda, researchers have prioritized the release
of all human genome sequencing information to ``maximize its benefit to
society'' ((19)). The meeting participants agreed that ``primary
sequence data should be rapidly released'', with ``sequence assemblies
{[}to{]} be released as soon as possible, in some centres{[}sic{]},
assemblies of greater than 1 kb would be released automatically on a
daily basis'', and that ``finished annotated sequence should be
submitted immediately to public databases'' ((19)). In 2003, another
meeting, held in Ft. Lauderdale, FL, re-affirmed the 1996 Bermuda
Principles, expanded upon them to apply more broadly towards sequencing
data, and called for further support of these practices ((19)).

These foundational ``Bermuda Principles'' and ``Ft. Launderdale
Accords'' agreements set the stage for both the Human Genome Project
(HGP) and the Human Microbiome Project (HMP) to generate and share
massive amounts of data over the course of their studies ((20), (21),
(22)). The goal of the HMP was to sequence all body sites to determine
the microbes found on and in the human body. Starting with major
projects such as the HGP and HMP, nucleic acid sequencing efforts have
been commonly uploaded and released using public databases. This allows
for researchers to use and re-use the data from the HGP and HMP. Us of
HMP sequencing data by researchers has resulted in over 650 scientific
publications ((23)), and the completion of metadata studies, including
those efforts participated in by these authors ((24), (25), (26)).

An important tool for creating phylogenies is the NCBI Basic Local
Alignment Search Tool (BLAST) ((27)), which is an essential tool for
comparative research. The BLAST algorithm allows users to compare a
nucleic acid or protein sequence to the NCBI database of over 1TB of
data to find similar and related sequences. Without the upload of
sequences to the NCBI database, the use and success of BLAST would not
be possible, despite the effort required on part of the researcher to
upload of sequences to one of the INSDC databases.

Availablity of data contributed to the rapid sequencing of the
SARS-CoV-2 virus during the 2020 pandemic and subsequent expedition of
vaccine development ((28)).

\subsection{Results}\label{results}

\begin{verbatim}
# A tibble: 154,720 x 79
   file                da    nsd   paper.x doi   doi_no_underscore journal_abrev
   <chr>               <chr> <chr> <chr>   <chr> <chr>             <chr>        
 1 Data/html/10.1128_~ <NA>  <NA>  https:~ 10.1~ 10.1128/aac.      aac          
 2 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 3 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 4 Data/html/10.1128_~ No    Yes   https:~ 10.1~ 10.1128/aac.0000~ aac          
 5 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 6 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 7 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 8 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
 9 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
10 Data/html/10.1128_~ No    No    https:~ 10.1~ 10.1128/aac.0000~ aac          
# i 154,710 more rows
# i 72 more variables: container.title <chr>, predicted <chr>,
#   alternative.id <chr>, created <date>, deposited <date>,
#   published.print <chr>, indexed <date>, issn <chr>, issue <dbl>,
#   issued <chr>, member <dbl>, page <chr>, prefix <dbl>, publisher <chr>,
#   score <dbl>, source <chr>, reference.count <dbl>, references.count <dbl>,
#   is.referenced.by.count <dbl>, title <chr>, type <chr>, ...
\end{verbatim}

\begin{verbatim}
[1] 12601
\end{verbatim}

\paragraph{Descriptive Statistics}\label{descriptive-statistics}

Using the Crossref database of DOIs, with validation from the Web of
Science, NCBI, and Scopus DOI databases, we downloaded 154720 unique
records of papers published in ASM journals. All papers were published
between 2000 and 2024. These papers came from \emph{Applied and
Envrionmental Microbiology (N = 20401), Antimicrobial Agents and
Chemotherapy (N = 21943), Infection and Immunity (N = 14500), Journal of
Bacteriology (N = 16821), Journal of Clinical Microbiology (N = 18450),
Journal of Microbiology and Biology Education (N = 1312), Journal of
Virology (N = 29797), Microbiology Resource Announcments} (formerly
known as \emph{Genome Announcements \texttt{\{r\}}}
\texttt{n\_journal{[}{[}3,2{]}{]}\ +\ n\_journal{[}{[}9,2{]}{]}}),
\emph{Microbiology Spectrum} \emph{(N = 6120),} \emph{mBio (N = 7732),
mSphere(N = 2712), and mSystems (N = 2331).}

\paragraph{Training Dataset}\label{training-dataset}

As 154720 is a large number of papers to investigate by hand, we created
a subset of the whole dataset to train two machine learning models. This
dataset was representative of the larger dataset in age of paper and
journal of origin and initially contained 500 papers. During iterative
model training, a subset of papers were hand validated after each
completed cycle. These subsets were created using dplyr's slice\_n() to
obtain papers from each journal. Additional hand selected papers were
validated to ensure robust model training. Validated papers were added
to the training set as we identified gaps in the dataset. This provided
a total of 1045 papers.

\begin{longtable}[]{@{}lrr@{}}
\caption{Trained Model Summary Statistics}\tabularnewline
\toprule\noalign{}
key & da\_model & nsd\_model \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
key & da\_model & nsd\_model \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
mtry & 300.00 & 200.00 \\
logLoss & 0.19 & 0.28 \\
AUC & 0.99 & 0.96 \\
prAUC & 0.95 & 0.95 \\
Accuracy & 0.95 & 0.90 \\
Kappa & 0.88 & 0.80 \\
F1 & 0.96 & 0.92 \\
Sensitivity & 0.96 & 0.94 \\
Specificity & 0.92 & 0.86 \\
Pos\_Pred\_Value & 0.96 & 0.90 \\
Neg\_Pred\_Value & 0.93 & 0.91 \\
Precision & 0.96 & 0.90 \\
Recall & 0.96 & 0.94 \\
Detection\_Rate & 0.62 & 0.53 \\
Balanced\_Accuracy & 0.94 & 0.90 \\
logLossSD & 0.02 & 0.02 \\
AUCSD & 0.01 & 0.01 \\
prAUCSD & 0.01 & 0.01 \\
AccuracySD & 0.01 & 0.02 \\
KappaSD & 0.03 & 0.04 \\
F1SD & 0.01 & 0.02 \\
SensitivitySD & 0.02 & 0.02 \\
SpecificitySD & 0.03 & 0.04 \\
Pos\_Pred\_ValueSD & 0.02 & 0.02 \\
Neg\_Pred\_ValueSD & 0.03 & 0.03 \\
PrecisionSD & 0.02 & 0.02 \\
RecallSD & 0.02 & 0.02 \\
Detection\_RateSD & 0.01 & 0.01 \\
Balanced\_AccuracySD & 0.02 & 0.02 \\
\end{longtable}

\paragraph{Random Forest Modeling}\label{random-forest-modeling}

Two random forest models were trained to predict if published scientific
papers ``contained new sequence data'', and if the paper ``had data
available'', one model for each variable. Each model was trained using
the same set of data, the normalized number of times a word or set of
words appears in each paper in the set. Briefly, the HTML content of
each paper was cleaned to removed non-meaningful words such as ``a, an,
the'', and separated into tokens, meaningful units of 1 - 3 words. These
tokens were also modified to eliminate issues with word tense. For
example ``interest\_importance'' was a two word token that appeared in
the model. These tokens were summed and normalized to each other and
were presented to the model as a table with each row as a paper, and
each column as the frequency of a specific token. See methods for more
information on this process.

Other models such as generalized linear regression (GLM) and boosted
trees (XGBoost) were evaluated, but were ultimately rejected in favor of
the random forest model due to fit (XX\textbf{data not shownXX}). Random
forest models were chosen by their high Area Under the Receiver Operator
Curve (AUROC) value to aid in this classification problem as the
creation of many decision trees helps to improve accuracy and precision.

These model tracings resulted in two different models to answer two
different questions. The new sequencing data model used an mtry value of
200 had an Area Under the Curve(AUC) of 0.96 and an accuracy of 0.9. The
sensitivity of the new sequencing data model was 0.94, and the
specificity of the model was 0.86. The data availability model used an
mtry value of 300 had an AUC of 0.99 and an accuracy of 0.95. The
sensitivity of the data availability model was 0.96, and the specificity
of the model was 0.92 (See Table 4XX for more information on trained
machine learning models). This shows that the models fit the data well,
and can provide classifications on new data with an expected error rate
of less than 10\%. We deemed this as acceptable, accounting for
variability in papers and data, as well as the large size of the dataset
on which we deployed the models.

\paragraph{Deploying on 150K+ Papers}\label{deploying-on-150k-papers}

After downloading the HTML content of each paper, we cleaned the HTML
content and readied it to apply our machine learning models to classify
each paper , in brief by removing non-meaningful words, followed by
counting and filtering tokens, and finishing in model-compatible matrix
format. Overall, 26.94\% of papers had new sequencing data. The journal
with the highest rate of new sequencing data was Genome Announcements at
98.72\%, and the lowest was Journal of Microbiology \& Biology Education
at 0.31\%. This was expected as \emph{Genome Announcements} and
\emph{Microbiology Resource Announcements} were the primary places for
the publication of new sequence data for ASM journals. In 2013,
\emph{Genome Announcements} was created to house papers describing new
genomic sequencing efforts, and these publications were redirected from
the \emph{Journal of Bacteriology}. In 2018, \emph{Genome Announcments}
was re-branded to \emph{Microbiology Resource Announcements}, the
permanent home of new sequencing efforts. This change in journal scope
for the \emph{Journal of Bacteriology} explained the change in
percentage of new sequencing data found in the journal over time. The
\emph{Journal of Microbiology and Biology Education}, as expected,
contained the lowest percentage of new sequencing data. The scope of
this journal is mainly resources for educators at the high school and
college levels, and we did not expect papers to contain new sequencing
data. In many of the ASM journals, the percentages of new sequencing
data did not change significantly over time as their scope did not
change. See Figure XX for percentages of new sequencing data for each
journal, and figure XXX for the trends over time. (XX? Put figures in
panel together for nsd overall and nsd over time? ) XX Other questions:
XX Why did AEM go up so much in 2013ish? XX Should we discuss the IAI
2012ish blurb? XX

\begin{figure}[H]

{\centering \pandocbounded{\includegraphics[keepaspectratio]{images/fract_nsd.png}}

}

\caption{Percentage of papers with new sequencing data}

\end{figure}%

\begin{figure}[H]

{\centering \pandocbounded{\includegraphics[keepaspectratio]{images/time_nsd.png}}

}

\caption{Percentage of papers wth new sequencing data over time}

\end{figure}%

Of papers with new sequencing data, 58.86\% of papers had data
available.

The journal with the highest rate of data availability was *Microbiology
Resource Announcements* at 99.83\%, and the lowest was *Journal of
Clinical Microbiology* at 13.44\%. This was expected as *Genome
Announcements* publishes mainly new genomic sequence data and makes the
data available. On average, papers in the dataset had a median of 25
citations/article. This number varies by journal, see table XXXX for
data by journal. The journal with the highest median rate of
citations/article was *Applied and Environmental Microbiology* at 38\%,
and the lowest was *Microbiology Resource Announcements* at 1\%. See
figures XXX and XXX for percentages of new sequencing data papers with
data availability for each journal and over time.

\paragraph{Training Dataset}\label{training-dataset-1}

We created a subset of the whole dataset to train our machine learning
models. The training dataset initially had N = 500 papers, but was
increased over time due to gaps in the dataset, and after subsequent
validation of the trained models (see below), a total of N = 1045. XXSee
Figure 1 for the distribution of papers per journal in the training
dataset.XX The journals in the dataset also span years 2000-2024. See
Figure 5XX for the distribution of papers per year in the whole and
training datasets. Overall, 43.73\% of papers had new sequencing data,
and 80.96\% of papers with new sequencing data, had data availability.
See Figures 3 and 4 for percentages of new sequencing data and data
availability for each journal. The journal with the highest rate of new
sequencing data was *Genome Announcements*at 96.67\%, and the lowest was
*Journal of Microbiology \& Biology Education* at 0\%. The journals with
the highest rate of data availability in new sequencing data papers were
*Genome Announcements and mSystems* at 100\%, and the lowest was
*Journal of Virology* at 28.57\%. On average, papers in the dataset had
median 10 citations/article. This number varies by journal, see table
XXXX for data by journal. The journal with the highest rate of
citations/article was *Infection and Immunity* at 40, and the lowest was
*Microbiology Resource Announcements*at 0.

\subsubsection{Year Published Distribution
Table}\label{year-published-distribution-table}

\begin{longtable}[]{@{}rrr@{}}
\caption{Distribution of Year Published for Whole and Training
Dataset}\tabularnewline
\toprule\noalign{}
year.published & n\_whole\_dataset & n\_training\_dataset \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
year.published & n\_whole\_dataset & n\_training\_dataset \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
2000 & 6009 & 30 \\
2001 & 5825 & 43 \\
2002 & 5807 & 23 \\
2003 & 6170 & 22 \\
2004 & 6461 & 21 \\
2005 & 6961 & 36 \\
2006 & 5873 & 29 \\
2007 & 5911 & 18 \\
2008 & 5476 & 12 \\
2009 & 5561 & 13 \\
2010 & 5622 & 21 \\
2011 & 6206 & 41 \\
2012 & 6609 & 24 \\
2013 & 6618 & 29 \\
2014 & 6875 & 31 \\
2015 & 6745 & 34 \\
2016 & 6417 & 38 \\
2017 & 5893 & 40 \\
2018 & 5899 & 40 \\
2019 & 5964 & 63 \\
2020 & 5886 & 65 \\
2021 & 6268 & 119 \\
2022 & 6824 & 70 \\
2023 & 6199 & 57 \\
2024 & 6212 & 116 \\
NA & 429 & 10 \\
\end{longtable}

\subsubsection{Descriptive Statisitcs about the Trained
Models}\label{descriptive-statisitcs-about-the-trained-models}

\paragraph{Figures for trained models}\label{figures-for-trained-models}

\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/auroc.new_seq_data.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/auroc.data_availability.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/hp_perf.rf.new_seq_data.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/hp_perf.rf.data_availability.png}}

\subsubsection{Regression Model using Negative Binomial
Models}\label{regression-model-using-negative-binomial-models}

In this study we sought to investigate the effect of new sequencing data
and data availability on the number of citations received by a given
paper. We focused on new sequencing data papers to determine the effect
of having data availability. This led us to the use of a negative
binomial regression model to best describe our data. All regression data
had new sequencing data (new sequencing data == ``Yes''). We focused on
the continuous outcome of ``number of citations'' with predictor
variables journal (categorical), age in months (continuous), and data
availability status (dichotomous). Due to the number of citations being
bell shaped with a long right tail (very few papers at advanced age with
many citations, producing a flat but non-zero line), the model that best
described our data was a negative binomial regression model. A negative
binomial model is appropriate for data that begins at zero and has a
long `tail' of data, as well as has differing means per group. This
model also includes a dispersion parameter, \(\theta\) to describe the
spread of the data. We applied a log transformation to our age variable
(age.in.months) to make linear the relationship between time and number
of citations received to help better describe the model relationship.

\[
N_(refs) = data availability + log(age.in.months) + journal + (journal*data availability) + (log(age.in.months)*data availability) + (journal*log(age.in.months)) + (log(age.in.months)*data availability*journal)
\]

See above for model formula, and supplemental table XXX6 for model
coefficients. In general, we found that new sequencing data papers that
made data availability received more citations over time than those that
did not. See figure XXXX for trends in each major journal. ??XX In
Figure 7XX, we have calculated the ratio of number of citations for
papers of similar age containing data vs those that do not.

\[
(Citations[data availability=Yes])/(Citations[data availability=No]) 
\]

Over time, papers with data availability receive more citations than
those without up to well over 1.5x in some journals (\emph{Journal of
Clinical Microbiology}). In all journals excepting the \emph{Journal of
Bacteriology}, papers with data availability have a greater number of
citations at time point 60 months after publication, if not sooner via
ratio plot. Over time this ratio increases further, demonstrating
increased citations for papers with data availability available over
time. Figure 8XX shows that the gap between predicted number of
citations for papers with data availability vs those without widens over
time, even beyond the width of the 95\% confidence interval.

\paragraph{Figures for Negative
Binomial}\label{figures-for-negative-binomial}

\pandocbounded{\includegraphics[keepaspectratio]{../Figures/negative_binomial/emmeans_contrast_plot.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/negative_binomial/model_predicted_plot.png}}

\subsection{Discussion}\label{discussion}

We investigated the impact of data availability on citation metrics in
new sequencing papers by deploying machine learning models on over
150,000 papers from the ASM journal program. Overall, making data
available increases citation metrics over time, an added benefit to
authors for the effort of making their data available.

On average, more than half of papers with new sequencing data had data
availability (58.86\%), showing that authors of new sequencing data
papers are more often than not, making their data publicly available.
This is in line with recent NIH policy requiring that data be made
available using the Data Management and Sharing Plan (DMS plan) outlined
in the NIH's NOT-OD-21-013 ((16)). This NIH policy went into effect in
2023. XXDo we want percent for 2024?XX Expectledly, journals
\emph{Genome Announcements} and \emph{Microbiology Resource
Announcements} had the highest rates of data availability in new
sequencing data papers. These journals publish primarily new genomic
sequences and are required by ASM to make data available. Journals such
as the \emph{Journal of Microbiology and Biology Education} and
\emph{Infection and Immunity} which publish fewer new sequencing data
papers due to their specific subject matters, have lower rates of new
sequencing data and therefore data availability in their journals.

Next, we looked further into the impact of data availability on citation
metrics using a negative binomial regression model. Using this model we
found that over time, papers with data availability receive more
citations than those without up to well over 1.5x the amount of
citations in some journals (Fig XXXX. \emph{Journal of Clinical
Microbiology}). These differences in ratios can be due to the fields of
papers found in each journal. Journals that are less likely to contain
sequencing papers include \emph{Infection and Immunity} as well as
\emph{Antimicrobial Agents and Chemotherapy}. XXadd more hereXX.

This effect intensifies over time, with the greatest differences in
citations occurring at the 108 months since publication time point. This
is great news to manuscript authors, that simply making their data
availability can provide as much as 50\% increase in citations over
time. We believe that this more than justifies the work of making data
available. We hope that these data will help to incentivize authors to
make data available.

We acknowledge the limitations of our study data, that by focusing only
on papers published in the ASM journal program, our results will not be
as generalizable. We understand that this relationship between data
availability and citation metrics may not be as strong in other families
of journals, but we hope with the newest NIH funding policies, these
trends will continue to improve data availability. Another limitation is
the availability of paper metadata from various databases, with Crossref
having the most complete metadata available for each paper. We were also
not able to track citation metrics for individual papers over time. Our
database sources only had available citation metrics at the time of
dataset download (February 2025), with no intermediate time points for
any given paper. This leaves us unable to understand trends over time
due to popularity or obsolescence of technique or results.

While making data availability in publications has a citation advantage,
we hope that is not the only reason authors choose to make their data
available. The need for reproducibility in science and demand for large
datasets are additional reasons, as well as the NIH funding requirement.
We hope that authors recognize the need to contribute to the data
``commons'', to further work done by others, and that even their
negative results have value and power to stop others from continuing
down dead ends.

\subsection{Materials and Methods}\label{materials-and-methods}

\begin{itemize}
\tightlist
\item
  need to put this stuff somewhere
\end{itemize}

Once these questions were answered, we moved to statistical analyses to
answer these and further questions, such as ``How does making my data
available impact my citation metrics over time?'' We were interested in
citation metrics as a concrete metric to examine how making data
availability benefits researchers and as a possible incentive towards
making data availability. Our hypotheses were that microbiologists would
have fairy high rates of data availability given the field's reliance on
comparative research, and that other fields such as immunology would
have lowered rates of data availability, as well as the hope that papers
with data availability would have a greater number of citations.

To avoid overfitting the models, we trained each model multiple times,
performing validations on a subset of data after each iteration. This
allowed us to have a greater number of papers in the training dataset by
adding these iteratively validated papers, as well as to have great
accuracy and precision within our models.

\subsubsection{Preparation of the Larger Experimental
Dataset}\label{preparation-of-the-larger-experimental-dataset}

To fully answer our research questions, we created a larger dataset with
N = 155779 papers curated from reference databases Crossref, NCBI,
Scopus, and the Web of Science ((29), (30) , (31), (32)). These papers
span all twelve ASM journals of interest from January 1st, 2000 to
December 31st, 2024. The ASM Journals of interest were \emph{Applied and
Envrionmental Microbiology; Antimicrobial Agents and Chemotherapy;
Infection and Immunity; Journal of Clinical Biology; Journal of
Virology; Journal of Bacteriology; Journal of Microbiology and Biology
Education; Microbiology Resource Announcments} (formerly known as
\emph{Genome Announcements}); \emph{mSystems; mSphere; mBio; and
Microbiology Spectrum.} The data was updated as of February 10th, 2025
with all citation counts frozen at that date.

\subsubsection{Creation of the Training
set}\label{creation-of-the-training-set}

To train our random forest machine learning model, we first created an
appropriate training data set. For our initial training set, we chose an
initial set of papers from across each journal and the time period of
interest, adding special emphasis to include papers that were part of
our desired set of interest (i.e.~contained published data) to ensure
that our two models could adequately characterize each paper as a new
sequencing paper and if it published raw sequencing data or not. After
creating our initial dataset, it was necessary to identify the status of
both variables by hand and determine if each paper contained ``new
sequencing data'', and if each one had ``data available''. This was
completed by opening each paper in an internet browser window, and
searching for a ``data availability'' or similar statement. See Table
1XXX for specific cases and how each of these cases were identified for
the purpose of this study.

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.4722}}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.2639}}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.2639}}@{}}
\caption{Possible Data Scenarios}\tabularnewline
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Scenario
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
new sequencing data Status
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
data availability Status
\end{minipage} \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Scenario
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
new sequencing data Status
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
data availability Status
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
Paper is not about generating new sequencing data & No & No \\
Paper is about generating new sequencing data but has no data available
& Yes & No \\
Paper is about generating new sequencing data and has data available &
Yes & Yes \\
Paper uses sequencing as a confirmation of experimental technique
(i.e.~confirmation of plasmid insertion) & No & No \\
Paper discusses new computational or experimental tools & No & No \\
Paper has microarray data & No & No \\
Papers using MLST ONLY & No & No \\
Papers using qPCR ONLY & No & No \\
Papers about protein sequencing that have nucleotide sequencing & Yes &
Yes/No depending on data availability \\
Papers using iRNA techniques & No & No \\
Papers using pyrosequencing/454 techniques & Yes & Yes/No depending on
data availability \\
\end{longtable}

\subsubsection{Adding Additional Training Set
Papers}\label{adding-additional-training-set-papers}

After initial training of our random forest models, a random sampling of
papers was collected for each journal using dplyr's slice\_n() to audit
the efficacy of the models ((33)). To audit the efficacy of the models,
we hand identified the status of both variables of interest, new
sequencing data and data availability. We looked for weaknesses in the
models, and updated methodology to reflect important areas of interest.
For example, in 2023 the ASM journals changed their formatting to
include the data availability statement of a paper in a sidebar of the
webpage. We identified this by noticing that all papers from journal
\emph{Microbiology Resource Announcements} from 2023-2024 were
incorrectly characterized by the model as data availability = No.~The
sidebar of the webpage was not included in the text the model was
considering, and code had to be updated to include all sidebar data for
all papers. These improvements to the model created a larger and more
comprehensive training set of N = 1045. These validations allowed us to
create confusion matrices for each model. Confusion matrices for the
final version of each trained model are available in XXX
table(XXsupplement?).

\subsubsection{Creation of the Training Data from Training
dataset}\label{creation-of-the-training-data-from-training-dataset}

To perform the computational steps required for these experiments, we
used the python tool Snakemake ( (34) ), and the University of
Michigan's high performance computing cluster (see acknowledgements).
Using our selected papers from the training dataset, we downloaded the
entirety of each paper's source HTML using the command line tool wget.
This allowed us to use the source HTML multiple times for updated
analyses without the need to re-query the ASM webservers numerous times.
Next, we performed cleaning of the HTML using R packages rvest ((35)),
textstem( (36)), and xml2 ((37)) to get the desired portions of the
paper from the HTML including the abstract, the body of paper, all
tables and figures with captions, as well as the side panels for all
papers, but especially those containing the data availability statements
in papers published after the 2023 change in webpage format (see above).
Then we removed unnecessary text using R packages tm(text
manipulation)((38), (39)) and textstem ((36)), as well as converting all
text to lowercase, and the removal of digits and non-alphabetic
characters such as whitespace. To have the fewest number of unique
words, we lemmatized (sort words by grouping inflected or variant forms
of the same word) words to trace them back to their root words and
eliminate any possible issues with word tense. After this, we created
and counted our `tokens', phrases of up to 1-3 consecutive words from
the text of the paper using R package tokeinziers ((40)). Towards the
goal of the fewest meaningful number of words, we used the `Snowball'
((41)) dictionary of `stop words' to remove non-meaningful words such as
articles `a', `an', and `the'. We removed the `space' character with an
underscore in multi-word tokens for ease of processing, and created a
count table for the tokens in each paper.

Once the tokens in each paper were counted, we transformed the data into
a sparse matrix format useable by the R package mikropml ((42)), using R
packages caret and dplyr ((43), (33), (44)). Tokens were filtered to
those which appear in greater than one paper. This allows comparison
between papers by the model. We removed near zero variants (tokens with
frequency very close to zero) as well as collapsing perfectly correlated
tokens (tokens that always appear together) using R packages caret and
mikropml to reduce model complexity. The data was then simplified to
keep only the following variables; tokens, frequency, journal
information, and hand identified new sequencing data and data
availability variables. This simiplified sparse matrix data had the mean
and standard deviation calculcated and saved for the frequency of each
token to later apply a z-scoring method to future data to be predicted
by the model.

\subsubsection{Training of the data availability and new sequencing data
Models}\label{training-of-the-data-availability-and-new-sequencing-data-models}

We trained two random forest machine learning models using mikropml's
``run\_ml'' function, one to determine if a paper contained new sequence
data, and another to determine if the paper had data available. The
mikropml ``run\_ml'' function uses methodology described by *Topcuoglu
et al.*((45)) to split data for model training. Random forest models
have one hyperparameter to tune, the mtry value. We began with mtry
values of 100, 200, 300, 400, 500, and 600, to find peak hyperparameter
performance given \emph{N tokens}. We trained the models multiple times
in accordance with existing methodologies, first to find the optimal
Area Under the Receiver-Operator Curve (AUROC) value for each model with
N=100 seeds. Then to find the best mtry performance for each model, with
N=1 seed. Finally, with N=1 seed to train each final model for use on
experimental data.

\subsubsection{Deploying the RF Models}\label{deploying-the-rf-models}

Once our RF models were ready we applied the same steps to ready papers
for application of machine learning models as the model training
dataset. See above for descriptions of web scraping HTML, cleaning HTML,
removing unnecessary text, and creation of token count table for
application in each of the machine learning models to determine the new
sequencing data and data availability statuses for each paper. Once the
frequency count tables were prepared for each paper, a z-score was
applied using the saved data from each model appropriately, using the
formula XX
\(((Observed token frequency - Model token frequency \mu)/(model token frequency sd))\).
This z-scoring formula was applied to standardize the frequency of each
token. Only tokens included in the machine learning models were retained
in experimental datasets. Finally, each model was deployed on each paper
to determine its new sequencing data and data availability status.

\subsubsection{Statistical Methodology}\label{statistical-methodology}

After each random forest model was deployed on our experimental dataset,
we used a negative binomial regression model using the R package MASS
((46)). A negative binomial regression model allows us to investigate
data where group means are different than the overall dataset mean. A
log transformation of the time variable(age.in.months) was applied prior
to estimating the statistical regression model to correct for the nature
of time as compared to other variables in the dataset. After applying
the negative binomial model, we calculated ratios of the estimated
number of citations per paper over time by data availability status
using R package emmeans ((47)). We also used R package sjPlot to
estimate the 95\% CI for estimated citations over time for each journal
by data availability status ((48)).

\begin{itemize}
\tightlist
\item
  Supplemental Material file list (where applicable)
\item
  Acknowledgments

  \begin{itemize}
  \tightlist
  \item
    The authors acknowledge lab members C. Armour, A. Mason, M. Coden,
    S. Lucas, and K. Sovacool for help hand-classifying papers.
  \item
    \textbf{\emph{``This research was supported in part through
    computational resources and~services provided by Advanced Research
    Computing at the University of~Michigan, Ann Arbor.''}}
    \href{https://rrid.site/data/record/nlx_144509-1/SCR_027337/resolver?q=SCR_027337\%2A&l=SCR_027337\%2A&i=rrid:scr_027337}{ARC's
    RRID is: SCR\_027337}
  \end{itemize}
\item
  References
\item
  Figures/Tables/stats to make/get

  \begin{itemize}
  \tightlist
  \item
    table of conditions to add to the methods of classification?
  \end{itemize}
\end{itemize}

\phantomsection\label{refs}
\begin{CSLReferences}{0}{1}
\bibitem[\citeproctext]{ref-congress2024}
\CSLLeftMargin{1. }%
\CSLRightInline{\href{https://www.congress.gov/crs-product/R47564}{Federal
{Research} and {Development} ({R}\&{D}) {Funding}: {FY2024}}.
legislation.}

\bibitem[\citeproctext]{ref-moniz_filedrawer_2025}
\CSLLeftMargin{2. }%
\CSLRightInline{\textbf{Moniz P}, \textbf{Druckman JN}, \textbf{Freese
J}. 2025. The file drawer problem in social science survey experiments.
Proceedings of the National Academy of Sciences
\textbf{122}:e2426937122.
doi:\href{https://doi.org/10.1073/pnas.2426937122}{10.1073/pnas.2426937122}.}

\bibitem[\citeproctext]{ref-rosenthal1979file}
\CSLLeftMargin{3. }%
\CSLRightInline{\textbf{Rosenthal R}. 1979. The file drawer problem and
tolerance for null results. Psychological bulletin \textbf{86}:638.}

\bibitem[\citeproctext]{ref-li_bigdata_2014}
\CSLLeftMargin{4. }%
\CSLRightInline{\textbf{Li Y}, \textbf{Chen L}. 2014. Big Biological
Data: Challenges and Opportunities. Genomics, Proteomics \&
Bioinformatics \textbf{12}:187--189.
doi:\href{https://doi.org/10.1016/j.gpb.2014.10.001}{10.1016/j.gpb.2014.10.001}.}

\bibitem[\citeproctext]{ref-pal_bigdata_2020}
\CSLLeftMargin{5. }%
\CSLRightInline{\textbf{Pal S}, \textbf{Mondal S}, \textbf{Das G},
\textbf{Khatua S}, \textbf{Ghosh Z}. 2020. Big data in biology: The hope
and present-day challenges in it. Gene Reports \textbf{21}:100869.
doi:\href{https://doi.org/10.1016/j.genrep.2020.100869}{10.1016/j.genrep.2020.100869}.}

\bibitem[\citeproctext]{ref-howe_2008}
\CSLLeftMargin{6. }%
\CSLRightInline{\textbf{Howe D}, \textbf{Costanzo M}, \textbf{Fey P},
\textbf{Gojobori T}, \textbf{Hannick L}, \textbf{Hide W}, \textbf{Hill
DP}, \textbf{Kania R}, \textbf{Schaeffer M}, \textbf{St Pierre S},
\textbf{Twigger S}, \textbf{White O}, \textbf{Yon Rhee S}. 2008. The
future of biocuration. Nature \textbf{455}:47--50.
doi:\href{https://doi.org/10.1038/455047a}{10.1038/455047a}.}

\bibitem[\citeproctext]{ref-not_data_reuse}
\CSLLeftMargin{7. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/notice-files/NOT-OD-24-096.html}{Expired
NOT-OD-24-096: Notice of special interest (NOSI): Promoting data reuse
for health research}.}

\bibitem[\citeproctext]{ref-par-23-2}
\CSLLeftMargin{8. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/pa-files/PAR-23-236.html}{PAR-23-236:
Early-stage biomedical data repositories and knowledgebases (R24
clinical trial not allowed)}.}

\bibitem[\citeproctext]{ref-indsc}
\CSLLeftMargin{9. }%
\CSLRightInline{\href{https://www.insdc.org/}{International nucleotide
sequence database collaboration}.}

\bibitem[\citeproctext]{ref-asm_relaunch_2021}
\CSLLeftMargin{10. }%
\CSLRightInline{\textbf{Cuomo CA}. 2021. The {Relaunch} of
{Microbiology} {Spectrum}. Microbiology Spectrum
\textbf{9}:10.1128/spectrum.00396--21.
doi:\href{https://doi.org/10.1128/spectrum.00396-21}{10.1128/spectrum.00396-21}.}

\bibitem[\citeproctext]{ref-force11}
\CSLLeftMargin{11. }%
\CSLRightInline{\href{https://force11.org/info/about-force11/}{About
{FORCE11} -- {FORCE11}}.}

\bibitem[\citeproctext]{ref-altman_2015}
\CSLLeftMargin{12. }%
\CSLRightInline{\textbf{Altman Director of Research and Head/Scientist,
Micah}, \textbf{Borgman Professor and Presidential Chair, Christine},
\textbf{Crosas Director of Data Science M}, \textbf{Matone Co-Director
M}. 2015. An introduction to the joint principles for data citation.
Bulletin of the Association for Information Science and Technology
\textbf{41}:43--45.
doi:\href{https://doi.org/10.1002/bult.2015.1720410313}{10.1002/bult.2015.1720410313}.}

\bibitem[\citeproctext]{ref-yilmaz_minimum_2011}
\CSLLeftMargin{13. }%
\CSLRightInline{\textbf{Yilmaz P}, \textbf{Field D}, \textbf{Knight R},
\textbf{Cole JR}, \textbf{Amaral-Zettler L}, \textbf{Gilbert JA},
\textbf{Karsch-Mizrachi I}, \textbf{Johnston A}, \textbf{Cochrane G},
\textbf{Vaughan R}, \textbf{Hunter C}, \textbf{Park J}, \textbf{Morrison
N}, \textbf{Rocca-Serra P}, \textbf{Sterk P}, \textbf{Arumugam M},
\textbf{Bailey M}, \textbf{Baumgartner L}, \textbf{Birren BW},
\textbf{Blaser MJ}, \textbf{Bonazzi V}, \textbf{Booth T}, \textbf{Bork
P}, \textbf{Bushman FD}, \textbf{Buttigieg PL}, \textbf{Chain PSG},
\textbf{Charlson E}, \textbf{Costello EK}, \textbf{Huot-Creasy H},
\textbf{Dawyndt P}, \textbf{DeSantis T}, \textbf{Fierer N},
\textbf{Fuhrman JA}, \textbf{Gallery RE}, \textbf{Gevers D},
\textbf{Gibbs RA}, \textbf{Gil IS}, \textbf{Gonzalez A}, \textbf{Gordon
JI}, \textbf{Guralnick R}, \textbf{Hankeln W}, \textbf{Highlander S},
\textbf{Hugenholtz P}, \textbf{Jansson J}, \textbf{Kau AL},
\textbf{Kelley ST}, \textbf{Kennedy J}, \textbf{Knights D},
\textbf{Koren O}, \textbf{Kuczynski J}, \textbf{Kyrpides N},
\textbf{Larsen R}, \textbf{Lauber CL}, \textbf{Legg T}, \textbf{Ley RE},
\textbf{Lozupone CA}, \textbf{Ludwig W}, \textbf{Lyons D},
\textbf{Maguire E}, \textbf{Meth BA}, \textbf{Meyer F}, \textbf{Muegge
B}, \textbf{Nakielny S}, \textbf{Nelson KE}, \textbf{Nemergut D},
\textbf{Neufeld JD}, \textbf{Newbold LK}, \textbf{Oliver AE},
\textbf{Pace NR}, \textbf{Palanisamy G}, \textbf{Peplies J},
\textbf{Petrosino J}, \textbf{Proctor L}, \textbf{Pruesse E},
\textbf{Quast C}, \textbf{Raes J}, \textbf{Ratnasingham S},
\textbf{Ravel J}, \textbf{Relman DA}, \textbf{Assunta-Sansone S},
\textbf{Schloss PD}, \textbf{Schriml L}, \textbf{Sinha R}, \textbf{Smith
MI}, \textbf{Sodergren E}, \textbf{Spor A}, \textbf{Stombaugh J},
\textbf{Tiedje JM}, \textbf{Ward DV}, \textbf{Weinstock GM},
\textbf{Wendel D}, \textbf{White O}, \textbf{Whiteley A}, \textbf{Wilke
A}, \textbf{Wortman JR}, \textbf{Yatsunenko T}, \textbf{Glckner FO}.
2011. Minimum information about a marker gene sequence ({MIMARKS}) and
minimum information about any (x) sequence ({MIxS}) specifications.
Nature Biotechnology \textbf{29}:415--420.
doi:\href{https://doi.org/10.1038/nbt.1823}{10.1038/nbt.1823}.}

\bibitem[\citeproctext]{ref-wilkinson_fair_2016}
\CSLLeftMargin{14. }%
\CSLRightInline{\textbf{Wilkinson MD}, \textbf{Dumontier M},
\textbf{Aalbersberg IjJ}, \textbf{Appleton G}, \textbf{Axton M},
\textbf{Baak A}, \textbf{Blomberg N}, \textbf{Boiten J-W}, \textbf{Silva
Santos LB da}, \textbf{Bourne PE}, \textbf{Bouwman J}, \textbf{Brookes
AJ}, \textbf{Clark T}, \textbf{Crosas M}, \textbf{Dillo I},
\textbf{Dumon O}, \textbf{Edmunds S}, \textbf{Evelo CT}, \textbf{Finkers
R}, \textbf{Gonzalez-Beltran A}, \textbf{Gray AJG}, \textbf{Groth P},
\textbf{Goble C}, \textbf{Grethe JS}, \textbf{Heringa J}, \textbf{Hoen
PAC 't}, \textbf{Hooft R}, \textbf{Kuhn T}, \textbf{Kok R}, \textbf{Kok
J}, \textbf{Lusher SJ}, \textbf{Martone ME}, \textbf{Mons A},
\textbf{Packer AL}, \textbf{Persson B}, \textbf{Rocca-Serra P},
\textbf{Roos M}, \textbf{Schaik R van}, \textbf{Sansone S-A},
\textbf{Schultes E}, \textbf{Sengstag T}, \textbf{Slater T},
\textbf{Strawn G}, \textbf{Swertz MA}, \textbf{Thompson M}, \textbf{Lei
J van der}, \textbf{Mulligen E van}, \textbf{Velterop J},
\textbf{Waagmeester A}, \textbf{Wittenburg P}, \textbf{Wolstencroft K},
\textbf{Zhao J}, \textbf{Mons B}. 2016. The {FAIR} {Guiding}
{Principles} for scientific data management and stewardship. Scientific
Data \textbf{3}:160018.
doi:\href{https://doi.org/10.1038/sdata.2016.18}{10.1038/sdata.2016.18}.}

\bibitem[\citeproctext]{ref-mirzayi_reporting_2021}
\CSLLeftMargin{15. }%
\CSLRightInline{\textbf{Mirzayi C}, \textbf{Renson A}, \textbf{Zohra F},
\textbf{Elsafoury S}, \textbf{Geistlinger L}, \textbf{Kasselman LJ},
\textbf{Eckenrode K}, \textbf{Wijgert J van de}, \textbf{Loughman A},
\textbf{Marques FZ}, \textbf{MacIntyre DA}, \textbf{Arumugam M},
\textbf{Azhar R}, \textbf{Beghini F}, \textbf{Bergstrom K},
\textbf{Bhatt A}, \textbf{Bisanz JE}, \textbf{Braun J}, \textbf{Bravo
HC}, \textbf{Buck GA}, \textbf{Bushman F}, \textbf{Casero D},
\textbf{Clarke G}, \textbf{Collado MC}, \textbf{Cotter PD},
\textbf{Cryan JF}, \textbf{Demmer RT}, \textbf{Devkota S},
\textbf{Elinav E}, \textbf{Escobar JS}, \textbf{Fettweis J},
\textbf{Finn RD}, \textbf{Fodor AA}, \textbf{Forslund S}, \textbf{Franke
A}, \textbf{Furlanello C}, \textbf{Gilbert J}, \textbf{Grice E},
\textbf{Haibe-Kains B}, \textbf{Handley S}, \textbf{Herd P},
\textbf{Holmes S}, \textbf{Jacobs JP}, \textbf{Karstens L},
\textbf{Knight R}, \textbf{Knights D}, \textbf{Koren O}, \textbf{Kwon
DS}, \textbf{Langille M}, \textbf{Lindsay B}, \textbf{McGovern D},
\textbf{McHardy AC}, \textbf{McWeeney S}, \textbf{Mueller NT},
\textbf{Nezi L}, \textbf{Olm M}, \textbf{Palm N}, \textbf{Pasolli E},
\textbf{Raes J}, \textbf{Redinbo MR}, \textbf{Rhlemann M},
\textbf{Balfour Sartor R}, \textbf{Schloss PD}, \textbf{Schriml L},
\textbf{Segal E}, \textbf{Shardell M}, \textbf{Sharpton T},
\textbf{Smirnova E}, \textbf{Sokol H}, \textbf{Sonnenburg JL},
\textbf{Srinivasan S}, \textbf{Thingholm LB}, \textbf{Turnbaugh PJ},
\textbf{Upadhyay V}, \textbf{Walls RL}, \textbf{Wilmes P},
\textbf{Yamada T}, \textbf{Zeller G}, \textbf{Zhang M}, \textbf{Zhao N},
\textbf{Zhang M}, \textbf{Zhao L}, \textbf{Zhao L}, \textbf{Bao W},
\textbf{Culhane A}, \textbf{Devanarayan V}, \textbf{Dopazo J},
\textbf{Fan X}, \textbf{Fischer M}, \textbf{Jones W}, \textbf{Kusko R},
\textbf{Mason CE}, \textbf{Mercer TR}, \textbf{Sansone S-A},
\textbf{Scherer A}, \textbf{Shi L}, \textbf{Thakkar S}, \textbf{Tong W},
\textbf{Wolfinger R}, \textbf{Hunter C}, \textbf{Segata N},
\textbf{Huttenhower C}, \textbf{Dowd JB}, \textbf{Jones HE},
\textbf{Waldron L}. 2021. Reporting guidelines for human microbiome
research: The {STORMS} checklist. Nature Medicine
\textbf{27}:1885--1892.
doi:\href{https://doi.org/10.1038/s41591-021-01552-x}{10.1038/s41591-021-01552-x}.}

\bibitem[\citeproctext]{ref-NIH_not-od-21-013_dms}
\CSLLeftMargin{16. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/notice-files/NOT-OD-21-013.html}{{NOT}-{OD}-21-013:
{Final} {NIH} {Policy} for {Data} {Management} and {Sharing}}.}

\bibitem[\citeproctext]{ref-Asm_opendata}
\CSLLeftMargin{17. }%
\CSLRightInline{\href{https://journals.asm.org/open-data-policy}{Open
{Data} {Policy}}. ASM Journals.}

\bibitem[\citeproctext]{ref-asm_ethics_2025}
\CSLLeftMargin{18. }%
\CSLRightInline{\href{https://journals.asm.org/publishing-ethics-procedures}{Publishing
{Ethics} {Policies} and {Procedures}}. ASM Journals.}

\bibitem[\citeproctext]{ref-maxsonjones2018}
\CSLLeftMargin{19. }%
\CSLRightInline{\textbf{Maxson Jones K}, \textbf{Ankeny RA},
\textbf{Cook-Deegan R}. 2018. The Bermuda Triangle: The Pragmatics,
Policies, and Principles for Data Sharing in the History of the Human
Genome Project. Journal of the History of Biology \textbf{51}:693--805.
doi:\href{https://doi.org/10.1007/s10739-018-9538-7}{10.1007/s10739-018-9538-7}.}

\bibitem[\citeproctext]{ref-proctor_integrative_2019}
\CSLLeftMargin{20. }%
\CSLRightInline{\textbf{Proctor LM}, \textbf{Creasy HH},
\textbf{Fettweis JM}, \textbf{Lloyd-Price J}, \textbf{Mahurkar A},
\textbf{Zhou W}, \textbf{Buck GA}, \textbf{Snyder MP}, \textbf{Strauss
JF}, \textbf{Weinstock GM}, \textbf{White O}, \textbf{Huttenhower C},
\textbf{The Integrative HMP (iHMP) Research Network Consortium}. 2019.
The {Integrative} {Human} {Microbiome} {Project}. Nature
\textbf{569}:641--648.
doi:\href{https://doi.org/10.1038/s41586-019-1238-8}{10.1038/s41586-019-1238-8}.}

\bibitem[\citeproctext]{ref-gevers_bioinformatics_2012}
\CSLLeftMargin{21. }%
\CSLRightInline{\textbf{Gevers D}, \textbf{Pop M}, \textbf{Schloss PD},
\textbf{Huttenhower C}. 2012. Bioinformatics for the {Human}
{Microbiome} {Project}. PLOS Computational Biology \textbf{8}:e1002779.
doi:\href{https://doi.org/10.1371/journal.pcbi.1002779}{10.1371/journal.pcbi.1002779}.}

\bibitem[\citeproctext]{ref-group_evaluation_2012}
\CSLLeftMargin{22. }%
\CSLRightInline{\textbf{Group JCHMPDGW}. 2012. Evaluation of {16S}
{rDNA}-{Based} {Community} {Profiling} for {Human} {Microbiome}
{Research}. PLOS ONE \textbf{7}:e39315.
doi:\href{https://doi.org/10.1371/journal.pone.0039315}{10.1371/journal.pone.0039315}.}

\bibitem[\citeproctext]{ref-HMP_2025}
\CSLLeftMargin{23. }%
\CSLRightInline{\href{https://commonfund.nih.gov/hmp}{Human {Microbiome}
{Project} ({HMP}) {\textbar} {NIH} {Common} {Fund}}.}

\bibitem[\citeproctext]{ref-ding_dynamics_2014}
\CSLLeftMargin{24. }%
\CSLRightInline{\textbf{Ding T}, \textbf{Schloss PD}. 2014. Dynamics and
associations of microbial community types across the human body. Nature
\textbf{509}:357--360.
doi:\href{https://doi.org/10.1038/nature13178}{10.1038/nature13178}.}

\bibitem[\citeproctext]{ref-abubucker_metabolic_2012}
\CSLLeftMargin{25. }%
\CSLRightInline{\textbf{Abubucker S}, \textbf{Segata N}, \textbf{Goll
J}, \textbf{Schubert AM}, \textbf{Izard J}, \textbf{Cantarel BL},
\textbf{Rodriguez-Mueller B}, \textbf{Zucker J}, \textbf{Thiagarajan M},
\textbf{Henrissat B}, \textbf{White O}, \textbf{Kelley ST},
\textbf{Meth B}, \textbf{Schloss PD}, \textbf{Gevers D},
\textbf{Mitreva M}, \textbf{Huttenhower C}. 2012. Metabolic
{Reconstruction} for {Metagenomic} {Data} and {Its} {Application} to the
{Human} {Microbiome}. PLOS Computational Biology \textbf{8}:e1002358.
doi:\href{https://doi.org/10.1371/journal.pcbi.1002358}{10.1371/journal.pcbi.1002358}.}

\bibitem[\citeproctext]{ref-huttenhower_structure_2012}
\CSLLeftMargin{26. }%
\CSLRightInline{\textbf{Huttenhower C}, \textbf{Gevers D},
\textbf{Knight R}, \textbf{Abubucker S}, \textbf{Badger JH},
\textbf{Chinwalla AT}, \textbf{Creasy HH}, \textbf{Earl AM},
\textbf{FitzGerald MG}, \textbf{Fulton RS}, \textbf{Giglio MG},
\textbf{Hallsworth-Pepin K}, \textbf{Lobos EA}, \textbf{Madupu R},
\textbf{Magrini V}, \textbf{Martin JC}, \textbf{Mitreva M},
\textbf{Muzny DM}, \textbf{Sodergren EJ}, \textbf{Versalovic J},
\textbf{Wollam AM}, \textbf{Worley KC}, \textbf{Wortman JR},
\textbf{Young SK}, \textbf{Zeng Q}, \textbf{Aagaard KM}, \textbf{Abolude
OO}, \textbf{Allen-Vercoe E}, \textbf{Alm EJ}, \textbf{Alvarado L},
\textbf{Andersen GL}, \textbf{Anderson S}, \textbf{Appelbaum E},
\textbf{Arachchi HM}, \textbf{Armitage G}, \textbf{Arze CA},
\textbf{Ayvaz T}, \textbf{Baker CC}, \textbf{Begg L}, \textbf{Belachew
T}, \textbf{Bhonagiri V}, \textbf{Bihan M}, \textbf{Blaser MJ},
\textbf{Bloom T}, \textbf{Bonazzi V}, \textbf{Paul Brooks J},
\textbf{Buck GA}, \textbf{Buhay CJ}, \textbf{Busam DA}, \textbf{Campbell
JL}, \textbf{Canon SR}, \textbf{Cantarel BL}, \textbf{Chain PSG},
\textbf{Chen I-MA}, \textbf{Chen L}, \textbf{Chhibba S}, \textbf{Chu K},
\textbf{Ciulla DM}, \textbf{Clemente JC}, \textbf{Clifton SW},
\textbf{Conlan S}, \textbf{Crabtree J}, \textbf{Cutting MA},
\textbf{Davidovics NJ}, \textbf{Davis CC}, \textbf{DeSantis TZ},
\textbf{Deal C}, \textbf{Delehaunty KD}, \textbf{Dewhirst FE},
\textbf{Deych E}, \textbf{Ding Y}, \textbf{Dooling DJ}, \textbf{Dugan
SP}, \textbf{Michael Dunne W}, \textbf{Scott Durkin A}, \textbf{Edgar
RC}, \textbf{Erlich RL}, \textbf{Farmer CN}, \textbf{Farrell RM},
\textbf{Faust K}, \textbf{Feldgarden M}, \textbf{Felix VM},
\textbf{Fisher S}, \textbf{Fodor AA}, \textbf{Forney LJ}, \textbf{Foster
L}, \textbf{Di Francesco V}, \textbf{Friedman J}, \textbf{Friedrich DC},
\textbf{Fronick CC}, \textbf{Fulton LL}, \textbf{Gao H}, \textbf{Garcia
N}, \textbf{Giannoukos G}, \textbf{Giblin C}, \textbf{Giovanni MY},
\textbf{Goldberg JM}, \textbf{Goll J}, \textbf{Gonzalez A},
\textbf{Griggs A}, \textbf{Gujja S}, \textbf{Kinder Haake S},
\textbf{Haas BJ}, \textbf{Hamilton HA}, \textbf{Harris EL},
\textbf{Hepburn TA}, \textbf{Herter B}, \textbf{Hoffmann DE},
\textbf{Holder ME}, \textbf{Howarth C}, \textbf{Huang KH}, \textbf{Huse
SM}, \textbf{Izard J}, \textbf{Jansson JK}, \textbf{Jiang H},
\textbf{Jordan C}, \textbf{Joshi V}, \textbf{Katancik JA},
\textbf{Keitel WA}, \textbf{Kelley ST}, \textbf{Kells C}, \textbf{King
NB}, \textbf{Knights D}, \textbf{Kong HH}, \textbf{Koren O},
\textbf{Koren S}, \textbf{Kota KC}, \textbf{Kovar CL}, \textbf{Kyrpides
NC}, \textbf{La Rosa PS}, \textbf{Lee SL}, \textbf{Lemon KP},
\textbf{Lennon N}, \textbf{Lewis CM}, \textbf{Lewis L}, \textbf{Ley RE},
\textbf{Li K}, \textbf{Liolios K}, \textbf{Liu B}, \textbf{Liu Y},
\textbf{Lo C-C}, \textbf{Lozupone CA}, \textbf{Dwayne Lunsford R},
\textbf{Madden T}, \textbf{Mahurkar AA}, \textbf{Mannon PJ},
\textbf{Mardis ER}, \textbf{Markowitz VM}, \textbf{Mavromatis K},
\textbf{McCorrison JM}, \textbf{McDonald D}, \textbf{McEwen J},
\textbf{McGuire AL}, \textbf{McInnes P}, \textbf{Mehta T},
\textbf{Mihindukulasuriya KA}, \textbf{Miller JR}, \textbf{Minx PJ},
\textbf{Newsham I}, \textbf{Nusbaum C}, \textbf{O'Laughlin M},
\textbf{Orvis J}, \textbf{Pagani I}, \textbf{Palaniappan K},
\textbf{Patel SM}, \textbf{Pearson M}, \textbf{Peterson J},
\textbf{Podar M}, \textbf{Pohl C}, \textbf{Pollard KS}, \textbf{Pop M},
\textbf{Priest ME}, \textbf{Proctor LM}, \textbf{Qin X}, \textbf{Raes
J}, \textbf{Ravel J}, \textbf{Reid JG}, \textbf{Rho M}, \textbf{Rhodes
R}, \textbf{Riehle KP}, \textbf{Rivera MC}, \textbf{Rodriguez-Mueller
B}, \textbf{Rogers Y-H}, \textbf{Ross MC}, \textbf{Russ C},
\textbf{Sanka RK}, \textbf{Sankar P}, \textbf{Fah Sathirapongsasuti J},
\textbf{Schloss JA}, \textbf{Schloss PD}, \textbf{Schmidt TM},
\textbf{Scholz M}, \textbf{Schriml L}, \textbf{Schubert AM},
\textbf{Segata N}, \textbf{Segre JA}, \textbf{Shannon WD}, \textbf{Sharp
RR}, \textbf{Sharpton TJ}, \textbf{Shenoy N}, \textbf{Sheth NU},
\textbf{Simone GA}, \textbf{Singh I}, \textbf{Smillie CS}, \textbf{Sobel
JD}, \textbf{Sommer DD}, \textbf{Spicer P}, \textbf{Sutton GG},
\textbf{Sykes SM}, \textbf{Tabbaa DG}, \textbf{Thiagarajan M},
\textbf{Tomlinson CM}, \textbf{Torralba M}, \textbf{Treangen TJ},
\textbf{Truty RM}, \textbf{Vishnivetskaya TA}, \textbf{Walker J},
\textbf{Wang L}, \textbf{Wang Z}, \textbf{Ward DV}, \textbf{Warren W},
\textbf{Watson MA}, \textbf{Wellington C}, \textbf{Wetterstrand KA},
\textbf{White JR}, \textbf{Wilczek-Boney K}, \textbf{Wu Y},
\textbf{Wylie KM}, \textbf{Wylie T}, \textbf{Yandava C}, \textbf{Ye L},
\textbf{Ye Y}, \textbf{Yooseph S}, \textbf{Youmans BP}, \textbf{Zhang
L}, \textbf{Zhou Y}, \textbf{Zhu Y}, \textbf{Zoloth L}, \textbf{Zucker
JD}, \textbf{Birren BW}, \textbf{Gibbs RA}, \textbf{Highlander SK},
\textbf{Meth BA}, \textbf{Nelson KE}, \textbf{Petrosino JF},
\textbf{Weinstock GM}, \textbf{Wilson RK}, \textbf{White O}, \textbf{The
Human Microbiome Project Consortium}. 2012. Structure, function and
diversity of the healthy human microbiome. Nature \textbf{486}:207--214.
doi:\href{https://doi.org/10.1038/nature11234}{10.1038/nature11234}.}

\bibitem[\citeproctext]{ref-altschul_blast}
\CSLLeftMargin{27. }%
\CSLRightInline{\textbf{Altschul SF}, \textbf{Gish W}, \textbf{Miller
W}, \textbf{Myers EW}, \textbf{Lipman DJ}. Basic {Local} {Alignment}
{Search} {Tool}.}

\bibitem[\citeproctext]{ref-maxmen_2021}
\CSLLeftMargin{28. }%
\CSLRightInline{\textbf{Maxmen A}. 2021. One million coronavirus
sequences: popular genome site hits mega milestone. Nature
\textbf{593}:21--21.
doi:\href{https://doi.org/10.1038/d41586-021-01069-w}{10.1038/d41586-021-01069-w}.}

\bibitem[\citeproctext]{ref-rcrossref}
\CSLLeftMargin{29. }%
\CSLRightInline{\textbf{Chamberlain S}, \textbf{Zhu H}, \textbf{Jahn N},
\textbf{Boettiger C}, \textbf{Ram K}. 2025.
\href{https://CRAN.R-project.org/package=rcrossref}{Rcrossref: Client
for various 'CrossRef' 'APIs'}.}

\bibitem[\citeproctext]{ref-sayers_ncbi_2020}
\CSLLeftMargin{30. }%
\CSLRightInline{\textbf{Sayers EW}, \textbf{Beck J}, \textbf{Brister
JR}, \textbf{Bolton EE}, \textbf{Canese K}, \textbf{Comeau DC},
\textbf{Funk K}, \textbf{Ketter A}, \textbf{Kim S}, \textbf{Kimchi A},
\textbf{Kitts PA}, \textbf{Kuznetsov A}, \textbf{Lathrop S}, \textbf{Lu
Z}, \textbf{McGarvey K}, \textbf{Madden TL}, \textbf{Murphy TD},
\textbf{O'Leary N}, \textbf{Phan L}, \textbf{Schneider VA},
\textbf{Thibaud-Nissen F}, \textbf{Trawick BW}, \textbf{Pruitt KD},
\textbf{Ostell J}. 2020. Database resources of the national center for
biotechnology information. Nucleic Acids Research \textbf{48}:D9--D16.
doi:\href{https://doi.org/10.1093/nar/gkz899}{10.1093/nar/gkz899}.}

\bibitem[\citeproctext]{ref-rscopus}
\CSLLeftMargin{31. }%
\CSLRightInline{\textbf{Muschelli J}. 2025.
\href{https://CRAN.R-project.org/package=rscopus}{Rscopus: Scopus
database 'API' interface}.}

\bibitem[\citeproctext]{ref-wos}
\CSLLeftMargin{32. }%
\CSLRightInline{Accessed 2025.
\href{https://www.webofscience.com/wos/author/author-search}{Web of
science}. Internet Database, Clarivate.}

\bibitem[\citeproctext]{ref-dplyr}
\CSLLeftMargin{33. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Franois R}, \textbf{Henry
L}, \textbf{Mller K}, \textbf{Vaughan D}. 2025.
\href{https://dplyr.tidyverse.org}{Dplyr: A grammar of data
manipulation}.}

\bibitem[\citeproctext]{ref-muxf6lder_snakemake}
\CSLLeftMargin{34. }%
\CSLRightInline{\textbf{Mlder F}, \textbf{Jablonski KP},
\textbf{Letcher B}, \textbf{Hall MB}, \textbf{Tomkins-Tinch CH},
\textbf{Sochat V}, \textbf{Forster J}, \textbf{Lee S}, \textbf{Twardziok
SO}, \textbf{Kanitz A}, \textbf{Wilm A}, \textbf{Holtgrewe M},
\textbf{Rahmann S}, \textbf{Nahnsen S}, \textbf{Kster J}. Sustainable
data analysis with Snakemake.
doi:\href{https://doi.org/10.12688/f1000research.29032.2}{10.12688/f1000research.29032.2}.}

\bibitem[\citeproctext]{ref-rvest}
\CSLLeftMargin{35. }%
\CSLRightInline{\textbf{Wickham H}. 2025.
\href{https://CRAN.R-project.org/package=rvest}{Rvest: Easily harvest
(scrape) web pages}.}

\bibitem[\citeproctext]{ref-textstem}
\CSLLeftMargin{36. }%
\CSLRightInline{\textbf{Rinker TW}. 2018.
\href{http://github.com/trinker/textstem}{{textstem}: Tools for stemming
and lemmatizing text}. Buffalo, New York.}

\bibitem[\citeproctext]{ref-xml2}
\CSLLeftMargin{37. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Hester J}, \textbf{Ooms J}.
2025. \href{https://CRAN.R-project.org/package=xml2}{xml2: Parse XML}.}

\bibitem[\citeproctext]{ref-tm_article}
\CSLLeftMargin{38. }%
\CSLRightInline{\textbf{Feinerer I}, \textbf{Hornik K}, \textbf{Meyer
D}. 2008. Text mining infrastructure in r. Journal of Statistical
Software \textbf{25}:1--54.
doi:\href{https://doi.org/10.18637/jss.v025.i05}{10.18637/jss.v025.i05}.}

\bibitem[\citeproctext]{ref-tm_manual}
\CSLLeftMargin{39. }%
\CSLRightInline{\textbf{Feinerer I}, \textbf{Hornik K}. 2025.
\href{https://CRAN.R-project.org/package=tm}{Tm: Text mining package}.}

\bibitem[\citeproctext]{ref-tokenizers}
\CSLLeftMargin{40. }%
\CSLRightInline{\textbf{Mullen LA}, \textbf{Benoit K}, \textbf{Keyes O},
\textbf{Selivanov D}, \textbf{Arnold J}. 2018. Fast, consistent
tokenization of natural language text. Journal of Open Source Software
\textbf{3}:655.
doi:\href{https://doi.org/10.21105/joss.00655}{10.21105/joss.00655}.}

\bibitem[\citeproctext]{ref-stopwords}
\CSLLeftMargin{41. }%
\CSLRightInline{\textbf{Benoit K}, \textbf{Muhr D}, \textbf{Watanabe K}.
2021. \href{https://CRAN.R-project.org/package=stopwords}{Stopwords:
Multilingual stopword lists}.}

\bibitem[\citeproctext]{ref-mikropml}
\CSLLeftMargin{42. }%
\CSLRightInline{\textbf{Topuolu BD}, \textbf{Lapp Z}, \textbf{Sovacool
KL}, \textbf{Snitkin E}, \textbf{Wiens J}, \textbf{Schloss PD}. 2021.
{mikropml}: User-friendly r package for supervised machine learning
pipelines. Journal of Open Source Software \textbf{6}:3073.
doi:\href{https://doi.org/10.21105/joss.03073}{10.21105/joss.03073}.}

\bibitem[\citeproctext]{ref-caret}
\CSLLeftMargin{43. }%
\CSLRightInline{\textbf{Kuhn}, \textbf{Max}. 2008. Building predictive
models in r using the caret package. Journal of Statistical Software
\textbf{28}:1--26.
doi:\href{https://doi.org/10.18637/jss.v028.i05}{10.18637/jss.v028.i05}.}

\bibitem[\citeproctext]{ref-tidyverse}
\CSLLeftMargin{44. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Averick M}, \textbf{Bryan
J}, \textbf{Chang W}, \textbf{McGowan LD}, \textbf{Franois R},
\textbf{Grolemund G}, \textbf{Hayes A}, \textbf{Henry L}, \textbf{Hester
J}, \textbf{Kuhn M}, \textbf{Pedersen TL}, \textbf{Miller E},
\textbf{Bache SM}, \textbf{Mller K}, \textbf{Ooms J}, \textbf{Robinson
D}, \textbf{Seidel DP}, \textbf{Spinu V}, \textbf{Takahashi K},
\textbf{Vaughan D}, \textbf{Wilke C}, \textbf{Woo K}, \textbf{Yutani H}.
2019. Welcome to the {tidyverse}. Journal of Open Source Software
\textbf{4}:1686.
doi:\href{https://doi.org/10.21105/joss.01686}{10.21105/joss.01686}.}

\bibitem[\citeproctext]{ref-topcuoglu_mikropml_2021}
\CSLLeftMargin{45. }%
\CSLRightInline{\textbf{Topuolu BD}, \textbf{Lapp Z}, \textbf{Sovacool
KL}, \textbf{Snitkin E}, \textbf{Wiens J}, \textbf{Schloss PD}. 2021.
Mikropml: {User}-{Friendly} {R} {Package} for {Supervised} {Machine}
{Learning} {Pipelines}. Journal of Open Source Software \textbf{6}:3073.
doi:\href{https://doi.org/10.21105/joss.03073}{10.21105/joss.03073}.}

\bibitem[\citeproctext]{ref-MASS}
\CSLLeftMargin{46. }%
\CSLRightInline{\textbf{Venables WN}, \textbf{Ripley BD}. 2002.
\href{https://www.stats.ox.ac.uk/pub/MASS4/}{Modern applied statistics
with s}Fourth. Springer, New York.}

\bibitem[\citeproctext]{ref-emmeans}
\CSLLeftMargin{47. }%
\CSLRightInline{\textbf{Lenth RV}. 2025.
\href{https://CRAN.R-project.org/package=emmeans}{Emmeans: Estimated
marginal means, aka least-squares means}.}

\bibitem[\citeproctext]{ref-sjPlot}
\CSLLeftMargin{48. }%
\CSLRightInline{\textbf{Ldecke D}. 2024.
\href{https://CRAN.R-project.org/package=sjPlot}{sjPlot: Data
visualization for statistics in social science}.}

\end{CSLReferences}




\end{document}
