% Options for packages loaded elsewhere
% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
  11pt,
]{article}
\usepackage{xcolor}
\usepackage[margin=1.0in]{geometry}
\usepackage{amsmath,amssymb}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage{setspace}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
% Make \paragraph and \subparagraph free-standing
\makeatletter
\ifx\paragraph\undefined\else
  \let\oldparagraph\paragraph
  \renewcommand{\paragraph}{
    \@ifstar
      \xxxParagraphStar
      \xxxParagraphNoStar
  }
  \newcommand{\xxxParagraphStar}[1]{\oldparagraph*{#1}\mbox{}}
  \newcommand{\xxxParagraphNoStar}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
  \let\oldsubparagraph\subparagraph
  \renewcommand{\subparagraph}{
    \@ifstar
      \xxxSubParagraphStar
      \xxxSubParagraphNoStar
  }
  \newcommand{\xxxSubParagraphStar}[1]{\oldsubparagraph*{#1}\mbox{}}
  \newcommand{\xxxSubParagraphNoStar}[1]{\oldsubparagraph{#1}\mbox{}}
\fi
\makeatother


\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\newsavebox\pandoc@box
\newcommand*\pandocbounded[1]{% scales image to fit in text height/width
  \sbox\pandoc@box{#1}%
  \Gscale@div\@tempa{\textheight}{\dimexpr\ht\pandoc@box+\dp\pandoc@box\relax}%
  \Gscale@div\@tempb{\linewidth}{\wd\pandoc@box}%
  \ifdim\@tempb\p@<\@tempa\p@\let\@tempa\@tempb\fi% select the smaller of both
  \ifdim\@tempa\p@<\p@\scalebox{\@tempa}{\usebox\pandoc@box}%
  \else\usebox{\pandoc@box}%
  \fi%
}
% Set default figure placement to htbp
\def\fps@figure{htbp}
\makeatother


% definitions for citeproc citations
\NewDocumentCommand\citeproctext{}{}
\NewDocumentCommand\citeproc{mm}{%
  \begingroup\def\citeproctext{#2}\cite{#1}\endgroup}
\makeatletter
 % allow citations to break across lines
 \let\@cite@ofmt\@firstofone
 % avoid brackets around text for \cite:
 \def\@biblabel#1{}
 \def\@cite#1#2{{#1\if@tempswa , #2\fi}}
\makeatother
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[2] % #1 hanging-indent, #2 entry-spacing
 {\begin{list}{}{%
  \setlength{\itemindent}{0pt}
  \setlength{\leftmargin}{0pt}
  \setlength{\parsep}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
   \setlength{\leftmargin}{\cslhangindent}
   \setlength{\itemindent}{-1\cslhangindent}
  \fi
  % set entry spacing
  \setlength{\itemsep}{#2\baselineskip}}}
 {\end{list}}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{\hfill\break\parbox[t]{\linewidth}{\strut\ignorespaces#1\strut}}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{\strut#1\strut}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{\strut#1\strut}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}



\setlength{\emergencystretch}{3em} % prevent overfull lines

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}



 


\usepackage{booktabs}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{xcolor}
\usepackage[left]{lineno}
\linenumbers
\modulolinenumbers
\usepackage{helvet}
\renewcommand*\familydefault{\sfdefault}
\usepackage[T1]{fontenc}
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\AtBeginDocument{%
\ifdefined\contentsname
  \renewcommand*\contentsname{Table of contents}
\else
  \newcommand\contentsname{Table of contents}
\fi
\ifdefined\listfigurename
  \renewcommand*\listfigurename{List of Figures}
\else
  \newcommand\listfigurename{List of Figures}
\fi
\ifdefined\listtablename
  \renewcommand*\listtablename{List of Tables}
\else
  \newcommand\listtablename{List of Tables}
\fi
\ifdefined\figurename
  \renewcommand*\figurename{Figure}
\else
  \newcommand\figurename{Figure}
\fi
\ifdefined\tablename
  \renewcommand*\tablename{Table}
\else
  \newcommand\tablename{Table}
\fi
}
\@ifpackageloaded{float}{}{\usepackage{float}}
\floatstyle{ruled}
\@ifundefined{c@chapter}{\newfloat{codelisting}{h}{lop}}{\newfloat{codelisting}{h}{lop}[chapter]}
\floatname{codelisting}{Listing}
\newcommand*\listoflistings{\listof{codelisting}{List of Listings}}
\makeatother
\makeatletter
\makeatother
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\@ifpackageloaded{subcaption}{}{\usepackage{subcaption}}
\makeatother
\usepackage{bookmark}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Data Accessibility Paper},
  pdfauthor={Joanna Colovas; Adena Collens; Patrick D. Schloss},
  pdfkeywords={data accessibility, data reproducibility, supervised
machine learning},
  colorlinks=true,
  linkcolor={blue},
  filecolor={Maroon},
  citecolor={Blue},
  urlcolor={Blue},
  pdfcreator={LaTeX via pandoc}}


\title{Data Accessibility Paper}
\author{Joanna Colovas \and Adena Collens \and Patrick D. Schloss}
\date{Oct 1, 2025}
\begin{document}
\maketitle


\setstretch{1.75}
\begin{itemize}
\item
  Abstract
\item
  Importance

  \begin{itemize}
  \tightlist
  \item
    Incentivize authors to publish/make available their original data
  \item
    Publishing data helps get more use out of research
  \item
    Helps eliminate file drawer effect as it shows negative data
  \end{itemize}
\item
  Keywords

  \begin{itemize}
  \tightlist
  \item
    Data accessibility
  \item
    Data reproducibility
  \end{itemize}
\end{itemize}

\subsection{Introduction}\label{introduction}

\subsubsection{Scientific Data as a Public
Good}\label{scientific-data-as-a-public-good}

The United States Government spent over two hundred million dollars
(USD) in 2024 on research expenditures ((1)). The result of all of these
investments are data, paid for in part by taxpayers. Therefore, data is
a public good. Public goods, for example public libraries, are able to
be used by anyone without barrier to entry, and without diminishing the
use of others. Data is available to be used by others, and best used as
for the benefit of those who provided the funds for it. Once data has
been generated, it can be used not only for initial analyses, but over
and over again in future studies or meta analyses. A key tenet of the
scientific method is the ability to replicate scientific findings to
ensure that they are not due to error. One way that scientific findings
can be replicated is by re-completing the same analyses by another
researcher. This is only possible if the data used to complete the
original analyses is available for use. Additionally, data can be used
to eliminate possible solutions to a problem by the publishing of
negative or non-significant data. Thinking of data as a public good, if
negative data is published, it can help researchers avoid sinking time
and financial resources into the investigation of non-viable hypotheses.
Unfortunately, researchers have few incentives to publish negative or
non-significant results. This lack of publication of non-fruitful
investigation is more commonly known as the ``file drawer effect'' ((2),
(\textbf{rosenthal\_1979?})).

Building on the file drawer effect, data availability (DA) is the
practice of making raw experimental data, such as nucleic acid
sequences, and analyses publicly accessible, often via upload in
maintained databases. These databases are public ``commons'' and help
widely benefit when they are well maintained and well used. With the
latest and greatest methodologies available across fields, increasing
amounts of data are being generated each day, especially in the
biological sciences ( (3), (4)). Availability of large quantities of
study data and metadata (data about data) is a necessary resource for
appropriate use and re-use of data and protocols as well as the
recreation of analyses. DA is a deeply important component of the
scientific process in the digital age, and curation of digital records
is a slowly emerging topic in data science( (\textbf{howe\_2008?})).
Available data and analyses are the gold standard for recreation of
studies and replication of their results. Not only is replication a
worthy goal, but large datasets are often underutilized, and can
continue to provide benefit and resources to researchers via their
re-use towards investigating and answering further questions. The
National Institutes of Health (NIH) has called for grant proposals for
the creation, enhancement, and maintenance, of new and existing data
repositories ( (5), (6)).

\subsubsection{Nucleic Acid Sequencing
Efforts}\label{nucleic-acid-sequencing-efforts}

Beginning in 1996 with the International Strategy Meeting on Human
Genome Sequencing in Bermuda, researchers have prioritized the release
of all human genome sequencing information so that it may ``maximize its
benefit to society'' ((7)). The meeting participants agreed that
``primary sequence data should be rapidly released'', with ``sequence
assemblies {[}to{]} be released as soon as possible, in some
centres{[}sic{]}, assemblies of greater than 1 kb would be released
automatically on a daily basis'', and that ``finished annotated sequence
should be submitted immediately to public databases'' ((7)). The
``Bermuda Principles,'' as they became known, have been embraced by the
large scale Human Genome Project (HGP) since 1998. The HGP endeavored to
publish the first complete human genome. In 2003, another meeting, held
in Ft. Lauderdale, FL, re-affirmed the 1996 Bermuda Principles, expanded
upon them to apply more broadly towards sequencing data, and called for
further support of these practices ((7)). These foundational agreements
set the stage for both the HGP and the Human Microbiome Project (HMP) to
generate and share massive amounts of data over the course of their
studies ((8), (9), (10)). The goal of the HMP was to sequence all body
sites to determine the microbes found on and in the human body.

Starting with major projects such as the HGP and HMP, nucleic acid
sequencing efforts have been commonly uploaded and released using public
databases. This allows for researchers to use and re-use the data from
the HGP and HMP. There are three major databases worldwide to support
sequencing and sharing efforts. The National Library of Medicine's (NLM)
National Center for Biotechnology (NCBI) in the United States, the
Research Organization of Information Systems' (ROIS) National Institute
of Genetics (NIG) in Japan, and the European Molecular Biology Lab's
(EMBL) European Bioinformatics Institue (EBI) in Europe. These three
databases are part of the International Nucleotide Sequence Database
Collaboration (INSDC) ((11)) . These large databases make comparative
research possible. Genetic lineages of microbes are determined by
creating phylogenetic trees which compare a new sequence to existing
sequences. Phylogenetic trees show how closely related a new microbial
genetic sequence is related to others studied before both in terms of
evolution and mutation and in structure and function. Comparative
genetics and genomics would not be possible without strong community
commitment to data availability.

\subsubsection{Current DA Policies}\label{current-da-policies}

Current data availability guidelines have been informed by a number of
policies created by funding agencies, peer-review journals, conference
and special task groups, as well as community interest groups. In 2011,
after the Future of Research Communication (FoRC) conference in Germany,
scientists and others came together to establish FORCE11, a community
interest group which seeks to encourage and promote data availability
standards ((12)) . Also in 2011, the Genomic Standards Consortium (GSC)
published a set of standards in \emph{Nature Biotechnology} to promote
the publication of the ``minimum information about a marker gene
sequence'' (MIMARKS) or ``minimum information about x sequence'' (MIxS)
((13)). These standards are checklists usable by data generators and
uploaders towards inclusion of relevant data with sequence uploads in
the International Nucleotide Sequence Database Collaboration (INSDC).
Some checklist items include if the data were published to an INSDC
database and metadata about the study systems, data collected, and
authors. An important factor is the ability to link the data to the
results and to the data generators.

In 2014 the FORCE11 group published the Joint Declaration of Data
Citation Principles (JDDCP), a document working towards the
standardization of data citation and its future availability((14)). Some
of the JDDCPs include crediting the authors of the data, providing data
with unique identifiers, and the persistence of the available data. The
Findable, Accessible, Interoperable, and Reuseable (FAIR) data science
guiding principles that were put forth in 2016 in \emph{Nature
Scientific Data} urges readers to ``improve the infrastructure
supporting the reuse of scholarly data'' ((15)). The FAIR principles are
often cited by NIH in funding calls for strong data science
practices((5)). In 2021, a \emph{Nature Medicine} publication put forth
the ``Stregthening the Organization and Reporting of Microbiome
Studies'' (STORMS) checklist to help authors identify report-worthy
elements of their data and metadata ( (16)). Some items on the STORMS
checklist include the sequencing method used in the study, the study
design, and physical location of the study. Unfortunately, none of these
DA principles or checklists are enforceable by any agency. The National
Institutes of Health (NIH) began enforcing the ``Policy for Data
Management and Sharing'' (NOT-OD-21-013) in January of 2023, requiring
all NIH funded studies to submit a data management and sharing plan
(DMS) with their funding applications, and comply with their DMS plan
after generation and publication of the funded work((17)). A DMS plan
includes detailed descriptions of data that will be generated in a
study, related tools, standards, and data preservation plans.
Non-compliance with NOT-OD-21-013 is identified by funding agencies
during annual Research Performance Progress Reports (RPPRs), and may
impact future funding decisions ((17)). We also believe that statements
of data ``available on request'' are not sufficient to be considered
available data. One example was published in \emph{Microbiome}, that a
reader in search of data may email the corresponding author, with
varying results ((18)).

With the advent of next generation sequencing, microbiology research has
generated large amounts of sequencing data, and it is common to upload
sequence data to a public repository as well as to include data in
research publications. Because of this, we set out to determine rates of
published raw data in the biological sciences, specifically
microbiology, by examining the American Society for Microbiology's (ASM)
library of published primary research journals. We were also interested
in the relationship between DA and number of citations received by a
published work.

\subsubsection{American Society for Microbiology Journals
(ASM)}\label{american-society-for-microbiology-journals-asm}

The American Society for Microbiology(ASM) is the major professional
body recognized by microbiologists. They have eighteen journals,
thirteen primary research journals, three review journals, and two
archive journals. In addition, several journals have been folded into
others or renamed over time. The ASM family of journals requires that
authors ``make data fully available, without restriction, except in rare
circumstances'' ((19)). They have adapted this policy from journals
\emph{Microbial Genomics} and \emph{PLOS}. In the ASM open data policy
they describe the use of a ``Data Availability Statement'' which
includes ``data description, name(s) of the repositories, and digital
object identifiers (DOIs) or accession numbers'' and encourages
publishing data on relevant public repositories ((19)). Consequences of
non-compliance to the ASM open data policy include contacting research
article authors to inform of non-compliance, publication of an
``Expression of Concern'' for the author and their compliance issues,
sanctions on publication in ASM journals, as well as contacting the
affiliated research institution and/or funding agencies of the authors
((20)). We endeavor to evaluate how well the microbiology community is
using reproducible data practices as we believe that this group of
researchers will be early adopters of the technologies available as a
result of both the ASM and NIH policies towards data availability.

\subsubsection{Data Avalability Case
Studies}\label{data-avalability-case-studies}

The availability of datasets also allows new questions to be answered
with existing data or the combination of multiple datasets, such as the
use of the Human Microbiome Project's (HMP) sequencing data by
researchers to create over 650 scientific publications ((21)), and the
completion of metadata studies, including those efforts participated in
by these authors ( (22), (23), (24)).

An important tool for creating phylogenies is the NCBI Basic Local
Alignment Search Tool (BLAST) ((\textbf{altschul1990?})). The BLAST
algorithm allows users to compare a nucleic acid or protein sequence to
the NCBI database of over 1TB of data to find similar and related
sequences. Without the upload of sequences to the NCBI database, the use
and success of BLAST would not be possible, despite the effort required
on part of the researcher to upload of sequences to one of the INSDC
databases.

Availablity of data contributed to the rapid sequencing of the
SARS-CoV-2 virus during the 2020 pandemic and subsequent expedition of
vaccine development ((25)).

With microbiologists commonly uploading nucleic acid sequences to public
databases, the aim of this study was to determine the current state of
data availability in twelve primary research journals from the ASM
family of journals. Primary research articles were classified with using
two machine learning models to answer two questions; ``Does this paper
contain new sequence data?'', and ``Is the data available?'' Once these
questions were answered, we moved to statistical analyses to answer
these and further questions, such as ``How does making my data available
impact my citation metrics over time?'' We were interested in citation
metrics as a concrete metric to examine how making DA benefits
researchers and as a possible incentive towards making DA.

\subsection{Results}\label{results}

\subsubsection{General Description of the
Experiment}\label{general-description-of-the-experiment}

We set out to determine the current state of data availability in twelve
primary research journals from the ASM family of journals. This
objective was completed by first acquiring all papers published in the
journals of interest between 2000 and 2024 using the Crossref database
and command line tools. We then trained two random forest machine
learning models to differentiate if each paper 1), contained ``New
Sequencing Data'' (NSD) and 2) if the paper had ``Data Available'' (DA).
To avoid overfitting the models, we trained each model multiple times,
performing validations on a subset of data after each iteration. This
allowed us to have a greater number of papers in the training dataset,
as well as to have great accuracy and precision within our models. Using
our trained models, we were able to classify over 150,000 papers from
the whole dataset to determine if they were NSD or contained DA. After
this, we could perform statistical modeling to describe the data, and
generate summary statistics. We were especially interested in the ways
in which NSD and DA impact citation metrics.

\subsubsection{ASM Journals}\label{asm-journals}

We used twelve of the ASM primary research journals in this study. Of
note, several journals had changes to their publication goals during the
2000-2024 time period. The \emph{Journal of Bacteriology} was the
primary journal to publish new genome announcements until 2013 when ASM
announced journal \emph{Genome Announcements} as a more permanent home
for this type of data. \emph{Genome Announcements} was active from 2013
until 2018, when it was re-branded to \emph{Microbiology Resource
Announcements}, which has been active from 2018 until present. These two
journals appear separately in our analyses due to the organization of
the Crossref database. Not all journals are equally likely to contain
NSD and have sequencing DA as a result of their field of interest.
Journals publishing new genome announcements have a high percentage of
NSD papers, and a high percentage of DA within those papers. As a
result, since the introduction of \emph{Genome Announcements} and
\emph{Microbiology Resource Announcements}, the \emph{Journal of
Bacteriology} has had fewer NSD papers, and fewer papers with DA since
2013. Another journal of note is \emph{Microbiology Spectrum} and its
re-brand. From 2013 until the fall of 2021, \emph{Microbiology Spectrum}
was a review journal. After this point, \emph{Microbiology Spectrum}
became a primary research journal ((26)). Review journals are less
likely to publish articles with NSD, and to have DA. Several journals,
including \emph{Microbiology Spectrum}, do not span the entire time
period for the study. Journals \emph{mBio} (b.2010), \emph{Microbiology
Spectrum} (b. 2013, re-brand 2021), \emph{mSphere} (b. 2016),
\emph{mSystems} (b. 2016), and \emph{Genome Announcements} (2013-2018).
Journals \emph{Antimicrobial Agents and Chemotherapy; Infection and
Immunity;} and the \emph{Journal of Microbiology and Biology Education;}
are all less likely to contain NSD and have DA than the other journals
in the dataset. Therefore, aggregate measures of all journal activity
together are thought to be less accurate than those depicting each
journal separately.

\subsubsection{Descriptive Statistics}\label{descriptive-statistics}

\paragraph{Whole Dataset}\label{whole-dataset}

list(container.title = c(``Antimicrobial Agents and Chemotherapy'',
``Applied and Environmental Microbiology'', ``Genome Announcements'',
``Infection and Immunity'', ``Journal of Bacteriology'', ``Journal of
Clinical Microbiology'', ``Journal of Microbiology \& Biology
Education'', ``Journal of Virology'', ``mBio'', ``Microbiology Resource
Announcements'', ``Microbiology Spectrum'', ``mSphere'', ``mSystems''),
n\_total = c(82, 92, 30, 76, 75, 77, 60, 87, 63, 155, 111, 57, 80),
n\_fract = c(7.84688995215311, 8.80382775119617, 2.87081339712919,
7.27272727272727, 7.17703349282297, 7.36842105263158, 5.74162679425837,
8.32535885167464, 6.02870813397129, 14.8325358851675, 10.622009569378,
5.45454545454545, 7.6555023923445), n\_nsd = c(23, 38, 29, 16, 25, 23,
0, 21, 17, 146, 49, 25, 45), fract\_nsd = c(28.0487804878049,
41.304347826087, 96.6666666666667, 21.0526315789474, 33.3333333333333,
29.8701298701299, 0, 24.1379310344828, 26.984126984127,
94.1935483870968, 44.1441441441441, 43.859649122807, 56.25), n\_da =
c(10, 28, 29, 9, 15, 14, 0, 6, 11, 145, 38, 20, 45), fract\_da =
c(43.4782608695652, 73.6842105263158, 100, 56.25, 60, 60.8695652173913,
NaN, 28.5714285714286, 64.7058823529412, 99.3150684931507,
77.5510204081633, 80, 100)), list(), , , list(x =
\textasciitilde fract\_nsd, y = \textasciitilde container.title),
list(), , , , , list(x = ``fract\_nsd'', y = ``container.title'')

list(container.title = c(``Antimicrobial Agents and Chemotherapy'',
``Applied and Environmental Microbiology'', ``Genome Announcements'',
``Infection and Immunity'', ``Journal of Bacteriology'', ``Journal of
Clinical Microbiology'', ``Journal of Microbiology \& Biology
Education'', ``Journal of Virology'', ``mBio'', ``Microbiology Resource
Announcements'', ``Microbiology Spectrum'', ``mSphere'', ``mSystems''),
n\_total = c(82, 92, 30, 76, 75, 77, 60, 87, 63, 155, 111, 57, 80),
n\_fract = c(7.84688995215311, 8.80382775119617, 2.87081339712919,
7.27272727272727, 7.17703349282297, 7.36842105263158, 5.74162679425837,
8.32535885167464, 6.02870813397129, 14.8325358851675, 10.622009569378,
5.45454545454545, 7.6555023923445), n\_nsd = c(23, 38, 29, 16, 25, 23,
0, 21, 17, 146, 49, 25, 45), fract\_nsd = c(28.0487804878049,
41.304347826087, 96.6666666666667, 21.0526315789474, 33.3333333333333,
29.8701298701299, 0, 24.1379310344828, 26.984126984127,
94.1935483870968, 44.1441441441441, 43.859649122807, 56.25), n\_da =
c(10, 28, 29, 9, 15, 14, 0, 6, 11, 145, 38, 20, 45), fract\_da =
c(43.4782608695652, 73.6842105263158, 100, 56.25, 60, 60.8695652173913,
NaN, 28.5714285714286, 64.7058823529412, 99.3150684931507,
77.5510204081633, 80, 100)), list(), , , list(x =
\textasciitilde fract\_da, y = \textasciitilde container.title), list(),
, , , , list(x = ``fract\_da'', y = ``container.title'')

list(container.title = c(``Antimicrobial Agents and Chemotherapy'',
``Applied and Environmental Microbiology'', ``Genome Announcements'',
``Infection and Immunity'', ``Journal of Bacteriology'', ``Journal of
Clinical Microbiology'', ``Journal of Microbiology \& Biology
Education'', ``Journal of Virology'', ``mBio'', ``Microbiology Resource
Announcements'', ``Microbiology Spectrum'', ``mSphere'', ``mSystems''),
n\_total = c(20297, 21853, 6714, 14490, 16806, 18421, 1305, 29761, 7705,
5878, 6119, 2615, 2327), n\_fract = c(13.1550122819866,
14.1634962505914, 4.35151758689749, 9.39134492614605, 10.8924046120642,
11.9391280113552, 0.845804356702594, 19.288876214426, 4.99381039723639,
3.80968429785276, 3.96588265031661, 1.69484934312436, 1.50818907130034),
n\_nsd = c(2765, 7261, 6628, 1287, 3734, 3713, 4, 3079, 2283, 5744,
2787, 971, 1430), fract\_nsd = c(13.622702862492, 33.2265592824784,
98.7190944295502, 8.88198757763975, 22.2182553849816, 20.156343303838,
0.306513409961686, 10.3457545109371, 29.6301103179753, 97.7203130316434,
45.5466579506455, 37.131931166348, 61.4525139664804), n\_da = c(897,
2546, 6546, 264, 1412, 499, 1, 972, 1473, 5734, 2174, 751, 1268),
fract\_da = c(32.4412296564195, 35.0640407657348, 98.7628243814122,
20.5128205128205, 37.8146759507231, 13.4392674387288, 25,
31.5686911334849, 64.5203679369251, 99.8259052924791, 78.0050233225691,
77.3429454170958, 88.6713286713287)), list(), , , list(x =
\textasciitilde fract\_nsd, y = \textasciitilde container.title),
list(), , , , , list(x = ``fract\_nsd'', y = ``container.title'')

list(container.title = c(``Antimicrobial Agents and Chemotherapy'',
``Applied and Environmental Microbiology'', ``Genome Announcements'',
``Infection and Immunity'', ``Journal of Bacteriology'', ``Journal of
Clinical Microbiology'', ``Journal of Microbiology \& Biology
Education'', ``Journal of Virology'', ``mBio'', ``Microbiology Resource
Announcements'', ``Microbiology Spectrum'', ``mSphere'', ``mSystems''),
n\_total = c(20297, 21853, 6714, 14490, 16806, 18421, 1305, 29761, 7705,
5878, 6119, 2615, 2327), n\_fract = c(13.1550122819866,
14.1634962505914, 4.35151758689749, 9.39134492614605, 10.8924046120642,
11.9391280113552, 0.845804356702594, 19.288876214426, 4.99381039723639,
3.80968429785276, 3.96588265031661, 1.69484934312436, 1.50818907130034),
n\_nsd = c(2765, 7261, 6628, 1287, 3734, 3713, 4, 3079, 2283, 5744,
2787, 971, 1430), fract\_nsd = c(13.622702862492, 33.2265592824784,
98.7190944295502, 8.88198757763975, 22.2182553849816, 20.156343303838,
0.306513409961686, 10.3457545109371, 29.6301103179753, 97.7203130316434,
45.5466579506455, 37.131931166348, 61.4525139664804), n\_da = c(897,
2546, 6546, 264, 1412, 499, 1, 972, 1473, 5734, 2174, 751, 1268),
fract\_da = c(32.4412296564195, 35.0640407657348, 98.7628243814122,
20.5128205128205, 37.8146759507231, 13.4392674387288, 25,
31.5686911334849, 64.5203679369251, 99.8259052924791, 78.0050233225691,
77.3429454170958, 88.6713286713287)), list(), , , list(x =
\textasciitilde fract\_da, y = \textasciitilde container.title), list(),
, , , , list(x = ``fract\_da'', y = ``container.title'')

Using the Crossref database of DOIs, with validation from the Web of
Science, NCBI, and Scopus DOI databases, we downloaded N = 154720 unique
records of papers published in ASM journals between 2000 and 2024. After
downloading the HTML content of each paper, we cleaned the HTML content
and readied it to apply our machine learning models to classify each
paper. Overall, 26.9429\% of papers had NSD, and 58.8615\% of papers
with NSD, had DA. See Figure 2XX for percentages of NSD and DA for each
journal. The journal with the highest rate of NSD was Genome
Announcements at 98.7191\%, and the lowest was Journal of Microbiology
\& Biology Education at 0.3065\%. The journal with the highest rate of
DA was *Microbiology Resource Announcements* at 99.8259\%, and the
lowest was *Journal of Clinical Microbiology* at 13.4393\%. This was
expected as *Genome Announcements* publishes mainly new genomic sequence
data and makes the data available. On average, papers in the dataset had
a median of 25 citations/article. This number varies by journal, see
table XXXX for data by journal. The journal with the highest median rate
of citations/article was *Applied and Environmental Microbiology* at
38\%, and the lowest was *Microbiology Resource Announcements* at 1\%.
The journals in the dataset span years 2000-2024. See table XXXX for the
distribution of papers per year in the dataset.

\paragraph{Training Dataset}\label{training-dataset}

We created a subset of the whole dataset to train our machine learning
models. The training dataset initially had N = 500 papers, but was
increased over time due to gaps in the dataset, and after subsequent
validation of the trained models (see below), a total of N = 1045. XXSee
Figure 1 for the distribution of papers per journal in the training
dataset.XX The journals in the dataset also span years 2000-2024. See
Figure 5XX for the distribution of papers per year in the whole and
training datasets. Overall, 43.7321\% of papers had NSD, and 80.9628\%
of papers with NSD, had DA. See Figures 3 and 4 for percentages of NSD
and DA for each journal. The journal with the highest rate of NSD was
*Genome Announcements*at 96.6667\%, and the lowest was *Journal of
Microbiology \& Biology Education* at 0\%. The journals with the highest
rate of DA in NSD papers were *Genome Announcements and mSystems* at
100\%, and the lowest was *Journal of Virology* at 28.5714\%. On
average, papers in the dataset had median 10 citations/article. This
number varies by journal, see table XXXX for data by journal. The
journal with the highest rate of citations/article was *Infection and
Immunity* at 40, and the lowest was *Microbiology Resource
Announcements*at 0.

\subsubsection{Year Published Distribution
Table}\label{year-published-distribution-table}

\begin{longtable}[]{@{}rrr@{}}
\caption{Distribution of Year Published for Whole and Training
Dataset}\tabularnewline
\toprule\noalign{}
year.published & n\_whole\_dataset & n\_training\_dataset \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
year.published & n\_whole\_dataset & n\_training\_dataset \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
2000 & 6009 & 30 \\
2001 & 5825 & 43 \\
2002 & 5807 & 23 \\
2003 & 6170 & 22 \\
2004 & 6461 & 21 \\
2005 & 6961 & 36 \\
2006 & 5873 & 29 \\
2007 & 5911 & 18 \\
2008 & 5476 & 12 \\
2009 & 5561 & 13 \\
2010 & 5622 & 21 \\
2011 & 6206 & 41 \\
2012 & 6609 & 24 \\
2013 & 6618 & 29 \\
2014 & 6875 & 31 \\
2015 & 6745 & 34 \\
2016 & 6417 & 38 \\
2017 & 5893 & 40 \\
2018 & 5899 & 40 \\
2019 & 5964 & 63 \\
2020 & 5886 & 65 \\
2021 & 6268 & 119 \\
2022 & 6824 & 70 \\
2023 & 6199 & 57 \\
2024 & 6212 & 116 \\
NA & 429 & 10 \\
\end{longtable}

list(year.published = c(2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007,
2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
2020, 2021, 2022, 2023, 2024, NA), n\_whole\_dataset = c(6009, 5825,
5807, 6170, 6461, 6961, 5873, 5911, 5476, 5561, 5622, 6206, 6609, 6618,
6875, 6745, 6417, 5893, 5899, 5964, 5886, 6268, 6824, 6199, 6212, 429),
n\_training\_dataset = c(30, 43, 23, 22, 21, 36, 29, 18, 12, 13, 21, 41,
24, 29, 31, 34, 38, 40, 40, 63, 65, 119, 70, 57, 116, 10)), list(), , ,
list(x = \textasciitilde year.published, y =
\textasciitilde n\_whole\_dataset), list(), , , , , list(x =
``year.published'', y = ``n\_whole\_dataset'')

\subsubsection{Descriptive Statisitcs about the Trained
Models}\label{descriptive-statisitcs-about-the-trained-models}

\begin{longtable}[]{@{}lrr@{}}
\caption{Trained Model Summary Statistics}\tabularnewline
\toprule\noalign{}
key & da\_model & nsd\_model \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
key & da\_model & nsd\_model \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
mtry & 300.0000 & 200.0000 \\
logLoss & 0.1864 & 0.2848 \\
AUC & 0.9878 & 0.9645 \\
prAUC & 0.9468 & 0.9465 \\
Accuracy & 0.9464 & 0.9036 \\
Kappa & 0.8826 & 0.8029 \\
F1 & 0.9586 & 0.9160 \\
Sensitivity & 0.9610 & 0.9350 \\
Specificity & 0.9199 & 0.8631 \\
Pos\_Pred\_Value & 0.9565 & 0.8984 \\
Neg\_Pred\_Value & 0.9290 & 0.9128 \\
Precision & 0.9565 & 0.8984 \\
Recall & 0.9610 & 0.9350 \\
Detection\_Rate & 0.6204 & 0.5257 \\
Balanced\_Accuracy & 0.9404 & 0.8991 \\
logLossSD & 0.0153 & 0.0206 \\
AUCSD & 0.0051 & 0.0107 \\
prAUCSD & 0.0131 & 0.0127 \\
AccuracySD & 0.0145 & 0.0189 \\
KappaSD & 0.0317 & 0.0388 \\
F1SD & 0.0112 & 0.0162 \\
SensitivitySD & 0.0163 & 0.0233 \\
SpecificitySD & 0.0294 & 0.0364 \\
Pos\_Pred\_ValueSD & 0.0152 & 0.0240 \\
Neg\_Pred\_ValueSD & 0.0277 & 0.0283 \\
PrecisionSD & 0.0152 & 0.0240 \\
RecallSD & 0.0163 & 0.0233 \\
Detection\_RateSD & 0.0105 & 0.0133 \\
Balanced\_AccuracySD & 0.0165 & 0.0199 \\
\end{longtable}

\paragraph{Figures for trained models}\label{figures-for-trained-models}

\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/auroc.new_seq_data.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/auroc.data_availability.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/hp_perf.rf.new_seq_data.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/ml_results/groundtruth/rf/hp_perf.rf.data_availability.png}}

Two random forest models were trained to predict if published scientific
papers ``contained new sequence data'' (NSD), and if the paper ``had
data available'' (DA), one model for each variable. Other models such as
generalized linear regression and boosted trees were explored, but were
ultimately discarded in favor of the random forest model (data not
shown). Random forest models were chosen to aid in this classification
problem as the creation of many decision trees helps to improve accuracy
and precision. This type of model has one hyperparameter, `mtry' or the
number of predictors to be sampled at each decision. During iterative
model training, a subset of papers were validated after each completed
training and deployment of each model. Papers from each journal, and
extras from certain journals were hand-validated against model
predictions to generate confusion matrices. Confusion matrices for the
final version of each trained model are available in YYYY
table(supplement?). The NSD model used an mtry value of 200 had an Area
Under the Curve(AUC) of 0.9645 and an accuracy of 0.9036. The
sensitivity of the NSD model was 0.935, and the specificity of the model
was 0.8631. The DA model used an mtry value of 300 had an AUC of 0.9878
and an accuracy of 0.9464. The sensitivity of the DA model was 0.961,
and the specificity of the model was 0.9199 (See table 4 for more
information on trained machine learning models). This shows that the
models fit the data well, and can provide classifications on new data
with an expected error rate of less than 10\%. We deemed this as
acceptable, accounting for variability in papers and data, as well as
the large size of the dataset on which we deployed the models.

\subsubsection{Regression Model using Negative Binomial
Models}\label{regression-model-using-negative-binomial-models}

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.7422}}
  >{\raggedleft\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.1016}}
  >{\raggedleft\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.1562}}@{}}
\caption{Coefficients for Negative Binomial Modeling}\tabularnewline
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
coefficients
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
full\_nsd\_yes
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
full\_nsd\_yes\_pvalue
\end{minipage} \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
coefficients
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
full\_nsd\_yes
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
full\_nsd\_yes\_pvalue
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
rsquared & 0.6914 & NA \\
(Intercept) & -1.1007 & 0.0000 \\
da\_factorYes & -1.6780 & 0.0000 \\
log(age.in.months) & 1.0193 & 0.0000 \\
container.titleApplied and Environmental Microbiology & -0.7625 &
0.0002 \\
container.titleGenome Announcements & -1.3780 & 0.6459 \\
container.titleInfection and Immunity & -0.3405 & 0.2905 \\
container.titleJournal of Bacteriology & -0.9194 & 0.0003 \\
container.titleJournal of Clinical Microbiology & 0.4635 & 0.0645 \\
container.titleJournal of Microbiology \& Biology Education & 3.0029 &
0.5597 \\
container.titleJournal of Virology & -0.3446 & 0.1051 \\
container.titlemBio & -1.4953 & 0.0000 \\
container.titleMicrobiology Resource Announcements & 2.3844 & 0.1270 \\
container.titleMicrobiology Spectrum & -3.3128 & 0.0000 \\
container.titlemSphere & -2.0889 & 0.0000 \\
container.titlemSystems & -3.0742 & 0.0000 \\
da\_factorYes:container.titleApplied and Environmental Microbiology &
0.8197 & 0.0063 \\
da\_factorYes:container.titleGenome Announcements & -1.0988 & 0.7162 \\
da\_factorYes:container.titleInfection and Immunity & 0.7932 & 0.0976 \\
da\_factorYes:container.titleJournal of Bacteriology & 1.8107 &
0.0000 \\
da\_factorYes:container.titleJournal of Clinical Microbiology & 0.6567 &
0.0923 \\
da\_factorYes:container.titleJournal of Microbiology \& Biology
Education & -20.8893 & 0.9989 \\
da\_factorYes:container.titleJournal of Virology & 1.3455 & 0.0000 \\
da\_factorYes:container.titlemBio & 1.0788 & 0.0015 \\
da\_factorYes:container.titleMicrobiology Resource Announcements &
-4.5437 & 0.0040 \\
da\_factorYes:container.titleMicrobiology Spectrum & 1.0664 & 0.0144 \\
da\_factorYes:container.titlemSphere & 1.1419 & 0.0277 \\
da\_factorYes:container.titlemSystems & 1.6684 & 0.0025 \\
da\_factorYes:log(age.in.months) & 0.4282 & 0.0000 \\
log(age.in.months):container.titleApplied and Environmental Microbiology
& 0.1606 & 0.0001 \\
log(age.in.months):container.titleGenome Announcements & -0.0672 &
0.9142 \\
log(age.in.months):container.titleInfection and Immunity & 0.0045 &
0.9407 \\
log(age.in.months):container.titleJournal of Bacteriology & 0.1231 &
0.0110 \\
log(age.in.months):container.titleJournal of Clinical Microbiology &
-0.1240 & 0.0100 \\
log(age.in.months):container.titleJournal of Microbiology \& Biology
Education & -1.2224 & 0.3434 \\
log(age.in.months):container.titleJournal of Virology & 0.0643 &
0.1261 \\
log(age.in.months):container.titlemBio & 0.4489 & 0.0000 \\
log(age.in.months):container.titleMicrobiology Resource Announcements &
-1.1018 & 0.0108 \\
log(age.in.months):container.titleMicrobiology Spectrum & 0.9146 &
0.0000 \\
log(age.in.months):container.titlemSphere & 0.5210 & 0.0000 \\
log(age.in.months):container.titlemSystems & 0.8673 & 0.0000 \\
da\_factorYes:log(age.in.months):container.titleApplied and
Environmental Microbiology & -0.1861 & 0.0037 \\
da\_factorYes:log(age.in.months):container.titleGenome Announcements &
0.1571 & 0.8029 \\
da\_factorYes:log(age.in.months):container.titleInfection and Immunity &
-0.2045 & 0.0431 \\
da\_factorYes:log(age.in.months):container.titleJournal of Bacteriology
& -0.5091 & 0.0000 \\
da\_factorYes:log(age.in.months):container.titleJournal of Clinical
Microbiology & -0.1082 & 0.1950 \\
da\_factorYes:log(age.in.months):container.titleJournal of Microbiology
\& Biology Education & NA & NA \\
da\_factorYes:log(age.in.months):container.titleJournal of Virology &
-0.3389 & 0.0000 \\
da\_factorYes:log(age.in.months):container.titlemBio & -0.2198 &
0.0045 \\
da\_factorYes:log(age.in.months):container.titleMicrobiology Resource
Announcements & 1.1682 & 0.0074 \\
da\_factorYes:log(age.in.months):container.titleMicrobiology Spectrum &
-0.2067 & 0.0953 \\
da\_factorYes:log(age.in.months):container.titlemSphere & -0.2635 &
0.0369 \\
da\_factorYes:log(age.in.months):container.titlemSystems & -0.4009 &
0.0040 \\
\end{longtable}

In this study we sought to investigate the effect of NSD and DA on the
number of citations received by a given paper. We focused on NSD papers
to determine the effect of having DA. This led us to the use of a
negative binomial regression model to best describe our data. All
regression data was NSD yes. We focused on the continuous outcome of
``number of citations'' with predictor variables journal (categorical),
age in months (continuous), and DA status (dichotomous). Due to the
number of citations being bell shaped with a long right tail (very few
papers at advanced age with many citations), the model that best
described our data was the negative binomial regression model. A
negative binomial model is appropriate for data that begins at zero and
has a long `tail' of data. This model also includes a dispersion
parameter to describe the spread of the data. We applied a log
transformation to our age variable (age.in.months) to better describe
the relationship between time and number of citations received. See
table XXX6? for model coefficients. In general, we found that NSD papers
that made DA received more citations over time than those that did not.
See figure YYYY for trends in each major journal. ??XX insert some kind
of metrics about \% more citations over time? per journal? ??XX Describe
figure emmeans contrast plot that i cannot get to insert in the document
for the life of me??? In all journals excepting the \emph{Journal of
Bacteriology}, papers with DA have a greater number of citations at time
point 60 months after publication, if not sooner via ratio plot. Over
time this ratio increases further, demonstrating increased citations for
papers with DA available over time.

\paragraph{Figures for Negative Binomial yay! come back to
theseXX}\label{figures-for-negative-binomial-yay-come-back-to-thesexx}

\pandocbounded{\includegraphics[keepaspectratio]{../Figures/negative_binomial/emmeans_contrast_plot.png}}
\pandocbounded{\includegraphics[keepaspectratio]{../Figures/negative_binomial/model_predicted_plot.png}}

\subsection{Discussion}\label{discussion}

We investigated the impact of data availability on citation metrics in
new sequencing papers by deploying machine learning models on over
150,000 papers from the ASM family of journals. Overall, making data
available increases citation metrics over time.

On average, more than half of papers with NSD had DA (58.8615\%),
showing that authors of NSD papers are more often than not, making their
data publicly available. This is in line with recent NIH policy
requiring that data be made available using the Data Management and
Sharing Plan (DMS plan) outlined in the NIH's NOT-OD-21-013 ((17)).
Expectledly, journals \emph{Genome Announcements} and \emph{Microbiology
Resource Announcements} had the highest rates of DA in NSD papers. These
journals publish primarily new genomic sequences and are required by ASM
to make data available. Journals such as the \emph{Journal of
Microbiology and Biology Education} and \emph{Infection and Immunity}
which publish fewer NSD papers, have lower rates of DA in their
journals.

Next, we looked further into the impact of DA on citation metrics using
a negative binomial regression model. Using this model we found that
over time, papers with DA receive more citations than those without up
to well over 1.5x in some journals (\emph{Journal of Clinical
Microbiology} ). This effect intensifies over time, with the greatest
differences in citations occurring at the 108 months since publication
time point. This is great news to manuscript authors, that simply making
DA can give as much as 50\% increase in citations over time. We believe
that this more than justifies the work of making data available. We hope
that these data will help to incentivize authors to make data available.

We acknowledge the limitations of our study data, focusing only on
papers published in the ASM family of journals. We understand that this
relationship between DA and citation metrics may not be as strong in
other families of journals, but we hope with the newest NIH funding
policies, these trends will continue. Another limitation is the
availability of paper metadata from various databases, with crossref's
metadata having the most complete metadata available for each paper. We
were also not able to see specific time points, only having access to
citation data for papers at time of dataset download (February 2025),
and no intermediate time points for any given paper.

While DA in publications has a citation advantage, we hope that is not
the only reason authors choose to make their data available. The need
for reproducibility in science and demand for large datasets are
additional reasons. We hope that authors recognize the need to
contribute to the data ``commons'', to further work done by others, and
that even their negative results have value and power to stop others
from continuing down dead ends.

\subsection{Materials and Methods}\label{materials-and-methods}

\subsubsection{Creation of the Training
set}\label{creation-of-the-training-set}

To train our random forest machine learning model, we first created an
appropriate training data set. Using the crossref database, we first
downloaded all papers from the selected ASM family of journals from the
time period beginning January 1st, 2000, and ending on December 31st,
2024. The data was updated as of February 10th, 2025 with all citation
counts frozen at that date. For our initial training set, we chose N =
500 papers from across each journal and time period, adding special
emphasis to include papers that were part of our desired set of interest
(i.e.~contained published data) to ensure that our two models could
adequately characterize each paper as a new sequencing paper and if it
published raw sequencing data or not. After creating our initial
dataset, it was necessary to identify the status of both variables by
hand and determine if each paper contained ``new sequencing data''
(NSD), and if each one had ``data available'' (DA). This was completed
by opening each paper in an internet browser window, and searching for a
``data availability'' or similar statement. See table XXX for specific
cases and how each of these cases were identified for the purpose of
this study.

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.4722}}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.2639}}
  >{\raggedright\arraybackslash}p{(\linewidth - 4\tabcolsep) * \real{0.2639}}@{}}
\caption{Possible Data Scenarios}\tabularnewline
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Scenario
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
NSD Status
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
DA Status
\end{minipage} \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Scenario
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
NSD Status
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
DA Status
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
Paper is not about generating new sequencing data & No & No \\
Paper is about generating new sequencing data but has no data available
& Yes & No \\
Paper is about generating new sequencing data and has data available &
Yes & Yes \\
Paper uses sequencing as a confirmation of experimental technique
(i.e.~confirmation of plasmid insertion) & No & No \\
Paper discusses new computational or experimental tools & No & No \\
Paper has microarray data & No & No \\
Papers using MLST ONLY & No & No \\
Papers using qPCR ONLY & No & No \\
Papers about protein sequencing that have nucleotide sequencing & Yes &
Yes/No depending on DA \\
Papers using iRNA techniques & No & No \\
Papers using pyrosequencing/454 techniques & Yes & Yes/No depending on
DA \\
\end{longtable}

\subsubsection{Adding Additional Training Set
Papers}\label{adding-additional-training-set-papers}

After initial trainings of our random forest models, a random sampling
of papers was collected for each journal to audit the efficacy of the
models. To audit the efficacy of the models, we hand identified the
status of both variables of interest, NSD and DA. We looked for
weaknesses in the models, and updated methodology to reflect important
areas of interest. For example, in 2023 the ASM journals changed their
formatting to include the data availability statement of a paper in a
sidebar of the webpage. We identified this by noticing that all papers
from journal \emph{Microbiology Resource Announcements} from 2023-2024
were incorrectly characterized by the model as DA = No.~The sidebar of
the webpage was not included in the text the model was considering, and
code had to be updated to include all sidebar data for all papers. These
improvements to the model created a larger and more comprehensive
training set of N = 1045. These validations allowed us to create
confusion matricies for each model. Confusion matricies for the final
version of each trained model are available in YYYY
table(XXsupplement?).

\subsubsection{Descriptive Statistics about the Training
Set}\label{descriptive-statistics-about-the-training-set}

\emph{I think this is covered earlier but i will leave it for now just
to make sure that i can move the sentences around if i need to}

There were XXX papers in the training data set from 12 ASM journals.
These papers came from journals \emph{Applied and Envrionmental
Microbiology; Antimicrobial Agents and Chemotherapy; Infection and
Immunity; Journal of Clinical Biology; Journal of Virology; Journal of
Bacteriology; Journal of Microbiology and Biology Education;
Microbiology Resource Announcments} (formerly known as \emph{Genome
Announcements}); \emph{mSystems; mSphere; mBio; and Microbiology
Spectrum.} See table XXXX for the number of papers from each journal in
the training dataset. The training dataset includes journal articles
published between January 1st, 2000 and December 31st, 2024. See table
XXXXX for the number of papers included from each year from 2000-2024.
XXX\% of training set data was NSD = Yes, and XXX\% of training set data
was DA = Yes.

\subsubsection{Creation of the Training Data from Training
dataset}\label{creation-of-the-training-data-from-training-dataset}

To perform the computational steps required for these experiments, we
used the python tool Snakemake ( (27) ), and the University of
Michigan's high performance computing cluster (see acknowledgements).
Using our selected papers from the training dataset, we downloaded the
entirety of each paper's source HTML using the command line tool wget.
This allowed us to use the source HTML multiple times for updated
analyses without the need to re-query the ASM webservers numerous times.
Next, we performed cleaning of the HTML using R packages rvest ((28))
and xml2 ((29)) to get the desired portions of the paper from the HTML
including the abstract, the body of paper, all tables and figures with
captions, as well as the side panels for all papers, but especially
those containing the data availability statements in papers published
after the 2023 change in webpage format (see above). Then we removed
unnecessary text using R packages tm(text manipulation)((30), (31)) and
textstem ((32)), as well as converting all text to lowercase, and the
removal of digits and non-alphabetic characters such as whitespace. To
have the fewest number of unique words, we lemmatized (sort words by
grouping inflected or variant forms of the same word) words to trace
them back to their root words and eliminate any possible issues with
word tense. After this, we created and counted our `tokens', phrases of
up to 1-3 consecutive words from the text of the paper using R package
tokeinziers ((\textbf{tokeinziers?})). Towards the goal of the fewest
meaningful number of words, we used the `Snowball' ((33)) dictionary of
`stop words' to remove non-meaningful words such as articles `a', `an',
and `the'. We removed the `space' character with an underscore in
multi-word tokens for ease of processing, and created a count table for
the tokens in each paper.

Once the tokens in each paper were counted, we transformed the data into
a sparse matrix format useable by the R package mikropml ((34) ref),
using R packages caret and dplyr ((35), (36), (37)). Tokens were
filtered to those which appear in greater than one paper. This allows
comparison between papers by the model. We removed near zero variants
(tokens with frequency very close to zero) as well as collapsing
perfectly correlated tokens (tokens that always appear together) using R
packages caret and mikropml to reduce model complexity. The data was
then simplified to keep only the following variables; tokens, frequency,
journal information, and hand identified NSD and DA variables. This
simiplified sparse matrix data had the mean and standard deviation
calculcated and saved for the frequency of each token to later apply a
z-scoring method to future data to be predicted by the model.

\subsubsection{Training of the DA and NSD
Models}\label{training-of-the-da-and-nsd-models}

We trained two random forest machine learning models using mikropml's
``run\_ml'' function, one to determine if a paper contained new sequence
data (NSD), and another to determine if the paper had data availabile
(DA). The mikropml ``run\_ml'' function uses methodology described by
*Topcuoglu et al.*((\textbf{topcuoglu2020?})) to split data for model
training. Random forest models have one hyperparameter to tune, the mtry
value. We began with mtry values of 100, 200, 300, 400, 500, and 600, to
find peak hyperparameter performance given \emph{N tokens}. We trained
the models multiple times in accordance with existing methodologies,
first to find the optimal Area Under the Receiver-Operator Curve (AUROC)
value for each model with N=100 seeds. Then to find the best mtry
performance for each model, with N=1 seed. Finally, with N=1 seed to
train each final model for use on experimental data.

\subsubsection{Preparation of the Experimental
Dataset}\label{preparation-of-the-experimental-dataset}

To fully answer our research questions, we created a larger database
with N = 155779 papers curated from reference datasets Crossref, NCBI,
Scopus, and the Web of Science ((\textbf{crossref?}), (\textbf{ncbi?}),
(\textbf{scopus?}), (38)). These papers span all twelve ASM journals of
interest from the start of 2000 to the end of 2024. Once database was
curated, we applied the same steps to ready papers for application of
machine learning models as the model training datasets. See above for
descriptions of webscraping html, cleaning html, removing unnecessary
text, and creation of token count table for application in each of the
machine learning models to determine the NSD and DA statuses for each
paper. Once the frequency count tables were prepared for each paper, a
z-score was applied using the saved data from each model appropriately,
using the formula XXX((observed\_token\_frequency -
model\_token\_frequency\_mean)/(model\_token\_frequency\_sd)). This
z-scoring formula was applied to standardize the frequency of each
token. Only tokens included in the machine learning models were retained
in experimental datasets. Finally, each model was deployed on each paper
to determine its NSD and DA status.

\subsubsection{Statistical Methodology}\label{statistical-methodology}

After each random forest model was deployed on our experimental dataset,
we used a negative binomial regression model using the R package MASS
((\textbf{MASSref?})). A negative binomial regression model allows us to
investigate data where group means are different than the overall
dataset mean. A log transformation of the time variable(age.in.months)
was applied prior to estimating the statistical regression model to
correct for the nature of time as compared to other variables in the
dataset. After applying the negative binomial model, we calculated
ratios of the estimated number of citations per paper over time by DA
status using R package emmeans ((39)). We also used R package sjPlot to
estimate the 95\% CI for estimated citations over time for each journal
by DA status ((\textbf{sjPlotref?})).

\begin{itemize}
\tightlist
\item
  Supplemental Material file list (where applicable)
\item
  Acknowledgments

  \begin{itemize}
  \item
    \textbf{\emph{``This research was supported in part through
    computational resources and~services provided by Advanced Research
    Computing at the University of~Michigan, Ann Arbor.''}}

    \href{https://rrid.site/data/record/nlx_144509-1/SCR_027337/resolver?q=SCR_027337\%2A&l=SCR_027337\%2A&i=rrid:scr_027337}{ARC's
    RRID is: SCR\_027337}
  \end{itemize}
\item
  References
\item
  Figures/Tables/stats to make/get

  \begin{itemize}
  \tightlist
  \item
    table of conditions to add to the methods of classification?
  \end{itemize}
\end{itemize}

\phantomsection\label{refs}
\begin{CSLReferences}{0}{1}
\bibitem[\citeproctext]{ref-congress2024}
\CSLLeftMargin{1. }%
\CSLRightInline{\href{https://www.congress.gov/crs-product/R47564}{Federal
{Research} and {Development} ({R}\&{D}) {Funding}: {FY2024}}.
legislation.}

\bibitem[\citeproctext]{ref-moniz_filedrawer_2025}
\CSLLeftMargin{2. }%
\CSLRightInline{\textbf{Moniz P}, \textbf{Druckman JN}, \textbf{Freese
J}. 2025. The file drawer problem in social science survey experiments.
Proceedings of the National Academy of Sciences
\textbf{122}:e2426937122.
doi:\href{https://doi.org/10.1073/pnas.2426937122}{10.1073/pnas.2426937122}.}

\bibitem[\citeproctext]{ref-li_bigdata_2014}
\CSLLeftMargin{3. }%
\CSLRightInline{\textbf{Li Y}, \textbf{Chen L}. 2014. Big Biological
Data: Challenges and Opportunities. Genomics, Proteomics \&
Bioinformatics \textbf{12}:187--189.
doi:\href{https://doi.org/10.1016/j.gpb.2014.10.001}{10.1016/j.gpb.2014.10.001}.}

\bibitem[\citeproctext]{ref-pal_bigdata_2020}
\CSLLeftMargin{4. }%
\CSLRightInline{\textbf{Pal S}, \textbf{Mondal S}, \textbf{Das G},
\textbf{Khatua S}, \textbf{Ghosh Z}. 2020. Big data in biology: The hope
and present-day challenges in it. Gene Reports \textbf{21}:100869.
doi:\href{https://doi.org/10.1016/j.genrep.2020.100869}{10.1016/j.genrep.2020.100869}.}

\bibitem[\citeproctext]{ref-not_data_reuse}
\CSLLeftMargin{5. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/notice-files/NOT-OD-24-096.html}{Expired
NOT-OD-24-096: Notice of special interest (NOSI): Promoting data reuse
for health research}.}

\bibitem[\citeproctext]{ref-par-23-2}
\CSLLeftMargin{6. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/pa-files/PAR-23-236.html}{PAR-23-236:
Early-stage biomedical data repositories and knowledgebases (R24
clinical trial not allowed)}.}

\bibitem[\citeproctext]{ref-maxsonjones2018}
\CSLLeftMargin{7. }%
\CSLRightInline{\textbf{Maxson Jones K}, \textbf{Ankeny RA},
\textbf{Cook-Deegan R}. 2018. The Bermuda Triangle: The Pragmatics,
Policies, and Principles for Data Sharing in the History of the Human
Genome Project. Journal of the History of Biology \textbf{51}:693--805.
doi:\href{https://doi.org/10.1007/s10739-018-9538-7}{10.1007/s10739-018-9538-7}.}

\bibitem[\citeproctext]{ref-proctor_integrative_2019}
\CSLLeftMargin{8. }%
\CSLRightInline{\textbf{Proctor LM}, \textbf{Creasy HH},
\textbf{Fettweis JM}, \textbf{Lloyd-Price J}, \textbf{Mahurkar A},
\textbf{Zhou W}, \textbf{Buck GA}, \textbf{Snyder MP}, \textbf{Strauss
JF}, \textbf{Weinstock GM}, \textbf{White O}, \textbf{Huttenhower C},
\textbf{The Integrative HMP (iHMP) Research Network Consortium}. 2019.
The {Integrative} {Human} {Microbiome} {Project}. Nature
\textbf{569}:641--648.
doi:\href{https://doi.org/10.1038/s41586-019-1238-8}{10.1038/s41586-019-1238-8}.}

\bibitem[\citeproctext]{ref-gevers_bioinformatics_2012}
\CSLLeftMargin{9. }%
\CSLRightInline{\textbf{Gevers D}, \textbf{Pop M}, \textbf{Schloss PD},
\textbf{Huttenhower C}. 2012. Bioinformatics for the {Human}
{Microbiome} {Project}. PLOS Computational Biology \textbf{8}:e1002779.
doi:\href{https://doi.org/10.1371/journal.pcbi.1002779}{10.1371/journal.pcbi.1002779}.}

\bibitem[\citeproctext]{ref-group_evaluation_2012}
\CSLLeftMargin{10. }%
\CSLRightInline{\textbf{Group JCHMPDGW}. 2012. Evaluation of {16S}
{rDNA}-{Based} {Community} {Profiling} for {Human} {Microbiome}
{Research}. PLOS ONE \textbf{7}:e39315.
doi:\href{https://doi.org/10.1371/journal.pone.0039315}{10.1371/journal.pone.0039315}.}

\bibitem[\citeproctext]{ref-indsc}
\CSLLeftMargin{11. }%
\CSLRightInline{\href{https://www.insdc.org/}{International nucleotide
sequence database collaboration}.}

\bibitem[\citeproctext]{ref-force11}
\CSLLeftMargin{12. }%
\CSLRightInline{\href{https://force11.org/info/about-force11/}{About
{FORCE11} -- {FORCE11}}.}

\bibitem[\citeproctext]{ref-yilmaz_minimum_2011}
\CSLLeftMargin{13. }%
\CSLRightInline{\textbf{Yilmaz P}, \textbf{Field D}, \textbf{Knight R},
\textbf{Cole JR}, \textbf{Amaral-Zettler L}, \textbf{Gilbert JA},
\textbf{Karsch-Mizrachi I}, \textbf{Johnston A}, \textbf{Cochrane G},
\textbf{Vaughan R}, \textbf{Hunter C}, \textbf{Park J}, \textbf{Morrison
N}, \textbf{Rocca-Serra P}, \textbf{Sterk P}, \textbf{Arumugam M},
\textbf{Bailey M}, \textbf{Baumgartner L}, \textbf{Birren BW},
\textbf{Blaser MJ}, \textbf{Bonazzi V}, \textbf{Booth T}, \textbf{Bork
P}, \textbf{Bushman FD}, \textbf{Buttigieg PL}, \textbf{Chain PSG},
\textbf{Charlson E}, \textbf{Costello EK}, \textbf{Huot-Creasy H},
\textbf{Dawyndt P}, \textbf{DeSantis T}, \textbf{Fierer N},
\textbf{Fuhrman JA}, \textbf{Gallery RE}, \textbf{Gevers D},
\textbf{Gibbs RA}, \textbf{Gil IS}, \textbf{Gonzalez A}, \textbf{Gordon
JI}, \textbf{Guralnick R}, \textbf{Hankeln W}, \textbf{Highlander S},
\textbf{Hugenholtz P}, \textbf{Jansson J}, \textbf{Kau AL},
\textbf{Kelley ST}, \textbf{Kennedy J}, \textbf{Knights D},
\textbf{Koren O}, \textbf{Kuczynski J}, \textbf{Kyrpides N},
\textbf{Larsen R}, \textbf{Lauber CL}, \textbf{Legg T}, \textbf{Ley RE},
\textbf{Lozupone CA}, \textbf{Ludwig W}, \textbf{Lyons D},
\textbf{Maguire E}, \textbf{Meth BA}, \textbf{Meyer F}, \textbf{Muegge
B}, \textbf{Nakielny S}, \textbf{Nelson KE}, \textbf{Nemergut D},
\textbf{Neufeld JD}, \textbf{Newbold LK}, \textbf{Oliver AE},
\textbf{Pace NR}, \textbf{Palanisamy G}, \textbf{Peplies J},
\textbf{Petrosino J}, \textbf{Proctor L}, \textbf{Pruesse E},
\textbf{Quast C}, \textbf{Raes J}, \textbf{Ratnasingham S},
\textbf{Ravel J}, \textbf{Relman DA}, \textbf{Assunta-Sansone S},
\textbf{Schloss PD}, \textbf{Schriml L}, \textbf{Sinha R}, \textbf{Smith
MI}, \textbf{Sodergren E}, \textbf{Spor A}, \textbf{Stombaugh J},
\textbf{Tiedje JM}, \textbf{Ward DV}, \textbf{Weinstock GM},
\textbf{Wendel D}, \textbf{White O}, \textbf{Whiteley A}, \textbf{Wilke
A}, \textbf{Wortman JR}, \textbf{Yatsunenko T}, \textbf{Glckner FO}.
2011. Minimum information about a marker gene sequence ({MIMARKS}) and
minimum information about any (x) sequence ({MIxS}) specifications.
Nature Biotechnology \textbf{29}:415--420.
doi:\href{https://doi.org/10.1038/nbt.1823}{10.1038/nbt.1823}.}

\bibitem[\citeproctext]{ref-altman_2015}
\CSLLeftMargin{14. }%
\CSLRightInline{\textbf{Altman Director of Research and Head/Scientist,
Micah}, \textbf{Borgman Professor and Presidential Chair, Christine},
\textbf{Crosas Director of Data Science M}, \textbf{Matone Co-Director
M}. 2015. An introduction to the joint principles for data citation.
Bulletin of the Association for Information Science and Technology
\textbf{41}:43--45.
doi:\href{https://doi.org/10.1002/bult.2015.1720410313}{10.1002/bult.2015.1720410313}.}

\bibitem[\citeproctext]{ref-wilkinson_fair_2016}
\CSLLeftMargin{15. }%
\CSLRightInline{\textbf{Wilkinson MD}, \textbf{Dumontier M},
\textbf{Aalbersberg IjJ}, \textbf{Appleton G}, \textbf{Axton M},
\textbf{Baak A}, \textbf{Blomberg N}, \textbf{Boiten J-W}, \textbf{Silva
Santos LB da}, \textbf{Bourne PE}, \textbf{Bouwman J}, \textbf{Brookes
AJ}, \textbf{Clark T}, \textbf{Crosas M}, \textbf{Dillo I},
\textbf{Dumon O}, \textbf{Edmunds S}, \textbf{Evelo CT}, \textbf{Finkers
R}, \textbf{Gonzalez-Beltran A}, \textbf{Gray AJG}, \textbf{Groth P},
\textbf{Goble C}, \textbf{Grethe JS}, \textbf{Heringa J}, \textbf{Hoen
PAC 't}, \textbf{Hooft R}, \textbf{Kuhn T}, \textbf{Kok R}, \textbf{Kok
J}, \textbf{Lusher SJ}, \textbf{Martone ME}, \textbf{Mons A},
\textbf{Packer AL}, \textbf{Persson B}, \textbf{Rocca-Serra P},
\textbf{Roos M}, \textbf{Schaik R van}, \textbf{Sansone S-A},
\textbf{Schultes E}, \textbf{Sengstag T}, \textbf{Slater T},
\textbf{Strawn G}, \textbf{Swertz MA}, \textbf{Thompson M}, \textbf{Lei
J van der}, \textbf{Mulligen E van}, \textbf{Velterop J},
\textbf{Waagmeester A}, \textbf{Wittenburg P}, \textbf{Wolstencroft K},
\textbf{Zhao J}, \textbf{Mons B}. 2016. The {FAIR} {Guiding}
{Principles} for scientific data management and stewardship. Scientific
Data \textbf{3}:160018.
doi:\href{https://doi.org/10.1038/sdata.2016.18}{10.1038/sdata.2016.18}.}

\bibitem[\citeproctext]{ref-mirzayi_reporting_2021}
\CSLLeftMargin{16. }%
\CSLRightInline{\textbf{Mirzayi C}, \textbf{Renson A}, \textbf{Zohra F},
\textbf{Elsafoury S}, \textbf{Geistlinger L}, \textbf{Kasselman LJ},
\textbf{Eckenrode K}, \textbf{Wijgert J van de}, \textbf{Loughman A},
\textbf{Marques FZ}, \textbf{MacIntyre DA}, \textbf{Arumugam M},
\textbf{Azhar R}, \textbf{Beghini F}, \textbf{Bergstrom K},
\textbf{Bhatt A}, \textbf{Bisanz JE}, \textbf{Braun J}, \textbf{Bravo
HC}, \textbf{Buck GA}, \textbf{Bushman F}, \textbf{Casero D},
\textbf{Clarke G}, \textbf{Collado MC}, \textbf{Cotter PD},
\textbf{Cryan JF}, \textbf{Demmer RT}, \textbf{Devkota S},
\textbf{Elinav E}, \textbf{Escobar JS}, \textbf{Fettweis J},
\textbf{Finn RD}, \textbf{Fodor AA}, \textbf{Forslund S}, \textbf{Franke
A}, \textbf{Furlanello C}, \textbf{Gilbert J}, \textbf{Grice E},
\textbf{Haibe-Kains B}, \textbf{Handley S}, \textbf{Herd P},
\textbf{Holmes S}, \textbf{Jacobs JP}, \textbf{Karstens L},
\textbf{Knight R}, \textbf{Knights D}, \textbf{Koren O}, \textbf{Kwon
DS}, \textbf{Langille M}, \textbf{Lindsay B}, \textbf{McGovern D},
\textbf{McHardy AC}, \textbf{McWeeney S}, \textbf{Mueller NT},
\textbf{Nezi L}, \textbf{Olm M}, \textbf{Palm N}, \textbf{Pasolli E},
\textbf{Raes J}, \textbf{Redinbo MR}, \textbf{Rhlemann M},
\textbf{Balfour Sartor R}, \textbf{Schloss PD}, \textbf{Schriml L},
\textbf{Segal E}, \textbf{Shardell M}, \textbf{Sharpton T},
\textbf{Smirnova E}, \textbf{Sokol H}, \textbf{Sonnenburg JL},
\textbf{Srinivasan S}, \textbf{Thingholm LB}, \textbf{Turnbaugh PJ},
\textbf{Upadhyay V}, \textbf{Walls RL}, \textbf{Wilmes P},
\textbf{Yamada T}, \textbf{Zeller G}, \textbf{Zhang M}, \textbf{Zhao N},
\textbf{Zhang M}, \textbf{Zhao L}, \textbf{Zhao L}, \textbf{Bao W},
\textbf{Culhane A}, \textbf{Devanarayan V}, \textbf{Dopazo J},
\textbf{Fan X}, \textbf{Fischer M}, \textbf{Jones W}, \textbf{Kusko R},
\textbf{Mason CE}, \textbf{Mercer TR}, \textbf{Sansone S-A},
\textbf{Scherer A}, \textbf{Shi L}, \textbf{Thakkar S}, \textbf{Tong W},
\textbf{Wolfinger R}, \textbf{Hunter C}, \textbf{Segata N},
\textbf{Huttenhower C}, \textbf{Dowd JB}, \textbf{Jones HE},
\textbf{Waldron L}. 2021. Reporting guidelines for human microbiome
research: The {STORMS} checklist. Nature Medicine
\textbf{27}:1885--1892.
doi:\href{https://doi.org/10.1038/s41591-021-01552-x}{10.1038/s41591-021-01552-x}.}

\bibitem[\citeproctext]{ref-NIH_not-od-21-013_dms}
\CSLLeftMargin{17. }%
\CSLRightInline{\href{https://grants.nih.gov/grants/guide/notice-files/NOT-OD-21-013.html}{{NOT}-{OD}-21-013:
{Final} {NIH} {Policy} for {Data} {Management} and {Sharing}}.}

\bibitem[\citeproctext]{ref-langille_available_2018}
\CSLLeftMargin{18. }%
\CSLRightInline{\textbf{Langille MGI}, \textbf{Ravel J}, \textbf{Fricke
WF}. 2018. {``{Available} upon request''}: Not good enough for
microbiome data! Microbiome \textbf{6}:8.
doi:\href{https://doi.org/10.1186/s40168-017-0394-z}{10.1186/s40168-017-0394-z}.}

\bibitem[\citeproctext]{ref-Asm_opendata}
\CSLLeftMargin{19. }%
\CSLRightInline{\href{https://journals.asm.org/open-data-policy}{Open
{Data} {Policy}}. ASM Journals.}

\bibitem[\citeproctext]{ref-asm_ethics_2025}
\CSLLeftMargin{20. }%
\CSLRightInline{\href{https://journals.asm.org/publishing-ethics-procedures}{Publishing
{Ethics} {Policies} and {Procedures}}. ASM Journals.}

\bibitem[\citeproctext]{ref-HMP_2025}
\CSLLeftMargin{21. }%
\CSLRightInline{\href{https://commonfund.nih.gov/hmp}{Human {Microbiome}
{Project} ({HMP}) {\textbar} {NIH} {Common} {Fund}}.}

\bibitem[\citeproctext]{ref-ding_dynamics_2014}
\CSLLeftMargin{22. }%
\CSLRightInline{\textbf{Ding T}, \textbf{Schloss PD}. 2014. Dynamics and
associations of microbial community types across the human body. Nature
\textbf{509}:357--360.
doi:\href{https://doi.org/10.1038/nature13178}{10.1038/nature13178}.}

\bibitem[\citeproctext]{ref-abubucker_metabolic_2012}
\CSLLeftMargin{23. }%
\CSLRightInline{\textbf{Abubucker S}, \textbf{Segata N}, \textbf{Goll
J}, \textbf{Schubert AM}, \textbf{Izard J}, \textbf{Cantarel BL},
\textbf{Rodriguez-Mueller B}, \textbf{Zucker J}, \textbf{Thiagarajan M},
\textbf{Henrissat B}, \textbf{White O}, \textbf{Kelley ST},
\textbf{Meth B}, \textbf{Schloss PD}, \textbf{Gevers D},
\textbf{Mitreva M}, \textbf{Huttenhower C}. 2012. Metabolic
{Reconstruction} for {Metagenomic} {Data} and {Its} {Application} to the
{Human} {Microbiome}. PLOS Computational Biology \textbf{8}:e1002358.
doi:\href{https://doi.org/10.1371/journal.pcbi.1002358}{10.1371/journal.pcbi.1002358}.}

\bibitem[\citeproctext]{ref-huttenhower_structure_2012}
\CSLLeftMargin{24. }%
\CSLRightInline{\textbf{Huttenhower C}, \textbf{Gevers D},
\textbf{Knight R}, \textbf{Abubucker S}, \textbf{Badger JH},
\textbf{Chinwalla AT}, \textbf{Creasy HH}, \textbf{Earl AM},
\textbf{FitzGerald MG}, \textbf{Fulton RS}, \textbf{Giglio MG},
\textbf{Hallsworth-Pepin K}, \textbf{Lobos EA}, \textbf{Madupu R},
\textbf{Magrini V}, \textbf{Martin JC}, \textbf{Mitreva M},
\textbf{Muzny DM}, \textbf{Sodergren EJ}, \textbf{Versalovic J},
\textbf{Wollam AM}, \textbf{Worley KC}, \textbf{Wortman JR},
\textbf{Young SK}, \textbf{Zeng Q}, \textbf{Aagaard KM}, \textbf{Abolude
OO}, \textbf{Allen-Vercoe E}, \textbf{Alm EJ}, \textbf{Alvarado L},
\textbf{Andersen GL}, \textbf{Anderson S}, \textbf{Appelbaum E},
\textbf{Arachchi HM}, \textbf{Armitage G}, \textbf{Arze CA},
\textbf{Ayvaz T}, \textbf{Baker CC}, \textbf{Begg L}, \textbf{Belachew
T}, \textbf{Bhonagiri V}, \textbf{Bihan M}, \textbf{Blaser MJ},
\textbf{Bloom T}, \textbf{Bonazzi V}, \textbf{Paul Brooks J},
\textbf{Buck GA}, \textbf{Buhay CJ}, \textbf{Busam DA}, \textbf{Campbell
JL}, \textbf{Canon SR}, \textbf{Cantarel BL}, \textbf{Chain PSG},
\textbf{Chen I-MA}, \textbf{Chen L}, \textbf{Chhibba S}, \textbf{Chu K},
\textbf{Ciulla DM}, \textbf{Clemente JC}, \textbf{Clifton SW},
\textbf{Conlan S}, \textbf{Crabtree J}, \textbf{Cutting MA},
\textbf{Davidovics NJ}, \textbf{Davis CC}, \textbf{DeSantis TZ},
\textbf{Deal C}, \textbf{Delehaunty KD}, \textbf{Dewhirst FE},
\textbf{Deych E}, \textbf{Ding Y}, \textbf{Dooling DJ}, \textbf{Dugan
SP}, \textbf{Michael Dunne W}, \textbf{Scott Durkin A}, \textbf{Edgar
RC}, \textbf{Erlich RL}, \textbf{Farmer CN}, \textbf{Farrell RM},
\textbf{Faust K}, \textbf{Feldgarden M}, \textbf{Felix VM},
\textbf{Fisher S}, \textbf{Fodor AA}, \textbf{Forney LJ}, \textbf{Foster
L}, \textbf{Di Francesco V}, \textbf{Friedman J}, \textbf{Friedrich DC},
\textbf{Fronick CC}, \textbf{Fulton LL}, \textbf{Gao H}, \textbf{Garcia
N}, \textbf{Giannoukos G}, \textbf{Giblin C}, \textbf{Giovanni MY},
\textbf{Goldberg JM}, \textbf{Goll J}, \textbf{Gonzalez A},
\textbf{Griggs A}, \textbf{Gujja S}, \textbf{Kinder Haake S},
\textbf{Haas BJ}, \textbf{Hamilton HA}, \textbf{Harris EL},
\textbf{Hepburn TA}, \textbf{Herter B}, \textbf{Hoffmann DE},
\textbf{Holder ME}, \textbf{Howarth C}, \textbf{Huang KH}, \textbf{Huse
SM}, \textbf{Izard J}, \textbf{Jansson JK}, \textbf{Jiang H},
\textbf{Jordan C}, \textbf{Joshi V}, \textbf{Katancik JA},
\textbf{Keitel WA}, \textbf{Kelley ST}, \textbf{Kells C}, \textbf{King
NB}, \textbf{Knights D}, \textbf{Kong HH}, \textbf{Koren O},
\textbf{Koren S}, \textbf{Kota KC}, \textbf{Kovar CL}, \textbf{Kyrpides
NC}, \textbf{La Rosa PS}, \textbf{Lee SL}, \textbf{Lemon KP},
\textbf{Lennon N}, \textbf{Lewis CM}, \textbf{Lewis L}, \textbf{Ley RE},
\textbf{Li K}, \textbf{Liolios K}, \textbf{Liu B}, \textbf{Liu Y},
\textbf{Lo C-C}, \textbf{Lozupone CA}, \textbf{Dwayne Lunsford R},
\textbf{Madden T}, \textbf{Mahurkar AA}, \textbf{Mannon PJ},
\textbf{Mardis ER}, \textbf{Markowitz VM}, \textbf{Mavromatis K},
\textbf{McCorrison JM}, \textbf{McDonald D}, \textbf{McEwen J},
\textbf{McGuire AL}, \textbf{McInnes P}, \textbf{Mehta T},
\textbf{Mihindukulasuriya KA}, \textbf{Miller JR}, \textbf{Minx PJ},
\textbf{Newsham I}, \textbf{Nusbaum C}, \textbf{O'Laughlin M},
\textbf{Orvis J}, \textbf{Pagani I}, \textbf{Palaniappan K},
\textbf{Patel SM}, \textbf{Pearson M}, \textbf{Peterson J},
\textbf{Podar M}, \textbf{Pohl C}, \textbf{Pollard KS}, \textbf{Pop M},
\textbf{Priest ME}, \textbf{Proctor LM}, \textbf{Qin X}, \textbf{Raes
J}, \textbf{Ravel J}, \textbf{Reid JG}, \textbf{Rho M}, \textbf{Rhodes
R}, \textbf{Riehle KP}, \textbf{Rivera MC}, \textbf{Rodriguez-Mueller
B}, \textbf{Rogers Y-H}, \textbf{Ross MC}, \textbf{Russ C},
\textbf{Sanka RK}, \textbf{Sankar P}, \textbf{Fah Sathirapongsasuti J},
\textbf{Schloss JA}, \textbf{Schloss PD}, \textbf{Schmidt TM},
\textbf{Scholz M}, \textbf{Schriml L}, \textbf{Schubert AM},
\textbf{Segata N}, \textbf{Segre JA}, \textbf{Shannon WD}, \textbf{Sharp
RR}, \textbf{Sharpton TJ}, \textbf{Shenoy N}, \textbf{Sheth NU},
\textbf{Simone GA}, \textbf{Singh I}, \textbf{Smillie CS}, \textbf{Sobel
JD}, \textbf{Sommer DD}, \textbf{Spicer P}, \textbf{Sutton GG},
\textbf{Sykes SM}, \textbf{Tabbaa DG}, \textbf{Thiagarajan M},
\textbf{Tomlinson CM}, \textbf{Torralba M}, \textbf{Treangen TJ},
\textbf{Truty RM}, \textbf{Vishnivetskaya TA}, \textbf{Walker J},
\textbf{Wang L}, \textbf{Wang Z}, \textbf{Ward DV}, \textbf{Warren W},
\textbf{Watson MA}, \textbf{Wellington C}, \textbf{Wetterstrand KA},
\textbf{White JR}, \textbf{Wilczek-Boney K}, \textbf{Wu Y},
\textbf{Wylie KM}, \textbf{Wylie T}, \textbf{Yandava C}, \textbf{Ye L},
\textbf{Ye Y}, \textbf{Yooseph S}, \textbf{Youmans BP}, \textbf{Zhang
L}, \textbf{Zhou Y}, \textbf{Zhu Y}, \textbf{Zoloth L}, \textbf{Zucker
JD}, \textbf{Birren BW}, \textbf{Gibbs RA}, \textbf{Highlander SK},
\textbf{Meth BA}, \textbf{Nelson KE}, \textbf{Petrosino JF},
\textbf{Weinstock GM}, \textbf{Wilson RK}, \textbf{White O}, \textbf{The
Human Microbiome Project Consortium}. 2012. Structure, function and
diversity of the healthy human microbiome. Nature \textbf{486}:207--214.
doi:\href{https://doi.org/10.1038/nature11234}{10.1038/nature11234}.}

\bibitem[\citeproctext]{ref-maxmen_2021}
\CSLLeftMargin{25. }%
\CSLRightInline{\textbf{Maxmen A}. 2021. One million coronavirus
sequences: popular genome site hits mega milestone. Nature
\textbf{593}:21--21.
doi:\href{https://doi.org/10.1038/d41586-021-01069-w}{10.1038/d41586-021-01069-w}.}

\bibitem[\citeproctext]{ref-asm_relaunch_2021}
\CSLLeftMargin{26. }%
\CSLRightInline{\textbf{Cuomo CA}. 2021. The {Relaunch} of
{Microbiology} {Spectrum}. Microbiology Spectrum
\textbf{9}:10.1128/spectrum.00396--21.
doi:\href{https://doi.org/10.1128/spectrum.00396-21}{10.1128/spectrum.00396-21}.}

\bibitem[\citeproctext]{ref-muxf6lder_snakemake}
\CSLLeftMargin{27. }%
\CSLRightInline{\textbf{Mlder F}, \textbf{Jablonski KP},
\textbf{Letcher B}, \textbf{Hall MB}, \textbf{Tomkins-Tinch CH},
\textbf{Sochat V}, \textbf{Forster J}, \textbf{Lee S}, \textbf{Twardziok
SO}, \textbf{Kanitz A}, \textbf{Wilm A}, \textbf{Holtgrewe M},
\textbf{Rahmann S}, \textbf{Nahnsen S}, \textbf{Kster J}. Sustainable
data analysis with Snakemake.
doi:\href{https://doi.org/10.12688/f1000research.29032.2}{10.12688/f1000research.29032.2}.}

\bibitem[\citeproctext]{ref-rvest}
\CSLLeftMargin{28. }%
\CSLRightInline{\textbf{Wickham H}. 2025.
\href{https://CRAN.R-project.org/package=rvest}{Rvest: Easily harvest
(scrape) web pages}.}

\bibitem[\citeproctext]{ref-xml2}
\CSLLeftMargin{29. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Hester J}, \textbf{Ooms J}.
2025. \href{https://CRAN.R-project.org/package=xml2}{xml2: Parse XML}.}

\bibitem[\citeproctext]{ref-tm_article}
\CSLLeftMargin{30. }%
\CSLRightInline{\textbf{Feinerer I}, \textbf{Hornik K}, \textbf{Meyer
D}. 2008. Text mining infrastructure in r. Journal of Statistical
Software \textbf{25}:1--54.
doi:\href{https://doi.org/10.18637/jss.v025.i05}{10.18637/jss.v025.i05}.}

\bibitem[\citeproctext]{ref-tm_manual}
\CSLLeftMargin{31. }%
\CSLRightInline{\textbf{Feinerer I}, \textbf{Hornik K}. 2025.
\href{https://CRAN.R-project.org/package=tm}{Tm: Text mining package}.}

\bibitem[\citeproctext]{ref-textstem}
\CSLLeftMargin{32. }%
\CSLRightInline{\textbf{Rinker TW}. 2018.
\href{http://github.com/trinker/textstem}{{textstem}: Tools for stemming
and lemmatizing text}. Buffalo, New York.}

\bibitem[\citeproctext]{ref-stopwords}
\CSLLeftMargin{33. }%
\CSLRightInline{\textbf{Benoit K}, \textbf{Muhr D}, \textbf{Watanabe K}.
2021. \href{https://CRAN.R-project.org/package=stopwords}{Stopwords:
Multilingual stopword lists}.}

\bibitem[\citeproctext]{ref-mikropml}
\CSLLeftMargin{34. }%
\CSLRightInline{\textbf{Topuolu BD}, \textbf{Lapp Z}, \textbf{Sovacool
KL}, \textbf{Snitkin E}, \textbf{Wiens J}, \textbf{Schloss PD}. 2021.
{mikropml}: User-friendly r package for supervised machine learning
pipelines. Journal of Open Source Software \textbf{6}:3073.
doi:\href{https://doi.org/10.21105/joss.03073}{10.21105/joss.03073}.}

\bibitem[\citeproctext]{ref-caret}
\CSLLeftMargin{35. }%
\CSLRightInline{\textbf{Kuhn}, \textbf{Max}. 2008. Building predictive
models in r using the caret package. Journal of Statistical Software
\textbf{28}:1--26.
doi:\href{https://doi.org/10.18637/jss.v028.i05}{10.18637/jss.v028.i05}.}

\bibitem[\citeproctext]{ref-dplyr}
\CSLLeftMargin{36. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Franois R}, \textbf{Henry
L}, \textbf{Mller K}, \textbf{Vaughan D}. 2025.
\href{https://dplyr.tidyverse.org}{Dplyr: A grammar of data
manipulation}.}

\bibitem[\citeproctext]{ref-tidyverse}
\CSLLeftMargin{37. }%
\CSLRightInline{\textbf{Wickham H}, \textbf{Averick M}, \textbf{Bryan
J}, \textbf{Chang W}, \textbf{McGowan LD}, \textbf{Franois R},
\textbf{Grolemund G}, \textbf{Hayes A}, \textbf{Henry L}, \textbf{Hester
J}, \textbf{Kuhn M}, \textbf{Pedersen TL}, \textbf{Miller E},
\textbf{Bache SM}, \textbf{Mller K}, \textbf{Ooms J}, \textbf{Robinson
D}, \textbf{Seidel DP}, \textbf{Spinu V}, \textbf{Takahashi K},
\textbf{Vaughan D}, \textbf{Wilke C}, \textbf{Woo K}, \textbf{Yutani H}.
2019. Welcome to the {tidyverse}. Journal of Open Source Software
\textbf{4}:1686.
doi:\href{https://doi.org/10.21105/joss.01686}{10.21105/joss.01686}.}

\bibitem[\citeproctext]{ref-wos}
\CSLLeftMargin{38. }%
\CSLRightInline{Accessed 2025.
\href{https://www.webofscience.com/wos/author/author-search}{Web of
science}. Internet Database, Clarivate.}

\bibitem[\citeproctext]{ref-emmeans}
\CSLLeftMargin{39. }%
\CSLRightInline{\textbf{Lenth RV}. 2025.
\href{https://CRAN.R-project.org/package=emmeans}{Emmeans: Estimated
marginal means, aka least-squares means}.}

\end{CSLReferences}




\end{document}
